export default {
  "10.1002/cpz1.90": " Xie, Z. et al. Gene Set Knowledge Discovery with Enrichr. Current Protocols vol. 1 (2021).",
  "10.1002/pro.3715": "Kanehisa, M. Toward understanding the origin and evolution of cellular organisms. Protein Science vol. 28 1947–1951 (2019).",
  "10.1016/j.cell.2017.10.049": "Subramanian, A. et al. A Next Generation Connectivity Map: L1000 Platform and the First 1,000,000 Profiles. Cell vol. 171 1437-1452.e17 (2017).",
  "10.1016/j.cell.2020.06.004": " Sanford, J. A. et al. Molecular Transducers of Physical Activity Consortium (MoTrPAC): Mapping the Dynamic Responses to Exercise. Cell vol. 181 1464–1474 (2020).",
  "10.1016/j.isci.2022.105799": " Petrosyan, V. et al. Identifying biomarkers of differential chemotherapy response in TNBC patient-derived xenografts with a CTD/WGCNA approach. iScience vol. 26 105799 (2023).",
  "10.1016/j.xgen.2023.100340": " Shapiro, J. A. et al. OpenPBTA: The Open Pediatric Brain Tumor Atlas. Cell Genomics 100340 (2023) doi:10.1016/j.xgen.2023.100340.",
  "10.1038/75556": "Ashburner, M. et al. Gene Ontology: tool for the unification of biology. Nature Genetics vol. 25 25–29 (2000).",
  "10.1038/nature11247": "An integrated encyclopedia of DNA elements in the human genome. Nature vol. 489 57–74 (2012).",
  "10.1038/ng.265": "Pannicke, U. et al. Reticular dysgenesis (aleukocytosis) is caused by mutations in the gene encoding mitochondrial adenylate kinase 2. Nature Genetics vol. 41 101–105 (2008).",
  "10.1038/ng.2653": "Lonsdale, J. et al. The Genotype-Tissue Expression (GTEx) project. Nature Genetics vol. 45 580–585 (2013).",
  "10.1038/s41467-018-03751-6": "Lachmann, A. et al. Massive mining of publicly available RNA-seq data from human and mouse. Nature Communications vol. 9 (2018).",
  "10.1038/s41598-020-72664-6": " Zhao, Y., Wong, L. & Goh, W. W. B. How to do quantile normalization correctly for gene expression data analyses. Scientific Reports vol. 10 (2020).",
  "10.1073/pnas.0506580102": "Subramanian, A. et al. Gene set enrichment analysis: A knowledge-based approach for interpreting genome-wide expression profiles. Proceedings of the National Academy of Sciences vol. 102 15545–15550 (2005).",
  "10.1093/bioinformatics/btac017": " Lelong, S. et al. BioThings SDK: a toolkit for building high-performance data APIs in biomedical research. Bioinformatics vol. 38 2077–2079 (2022).",
  "10.1093/glycob/cwz080": "York, W. S. et al. GlyGen: Computational and Informatics Resources for Glycoscience. Glycobiology vol. 30 72–73 (2019).",
  "10.1093/nar/gkaa1024": "Martens, M. et al. WikiPathways: connecting communities. Nucleic Acids Research vol. 49 D613–D621 (2020).",
  "10.1093/nar/gkaa1043": "Köhler, S. et al. The Human Phenotype Ontology in 2021. Nucleic Acids Research vol. 49 D1207–D1217 (2020).",
  "10.1093/nar/gkaa1083": "Blake, J. A. et al. Mouse Genome Database (MGD): Knowledgebase for mouse–human comparative biology. Nucleic Acids Research vol. 49 D981–D987 (2020).",
  "10.1093/nar/gkab359": " Kuleshov, M. V. et al. KEA3: improved kinase enrichment analysis via data integration. Nucleic Acids Research vol. 49 W304–W316 (2021).",
  "10.1093/nar/gkac1010": "Sollis, E. et al. The NHGRI-EBI GWAS Catalog: knowledgebase and deposition resource. Nucleic Acids Research vol. 51 D977–D985 (2022).",
  "10.1093/nar/gkac328": "Evangelista, J. E. et al. SigCom LINCS: data and metadata search engine for a million gene expression signatures. Nucleic Acids Research vol. 50 W697–W709 (2022).",
  "10.1093/nar/gkac956": " Kim, S. et al. PubChem 2023 update. Nucleic Acids Research vol. 51 D1373–D1380 (2022).",
  "10.1093/nar/gkac963": "Kanehisa, M., Furumichi, M., Sato, Y., Kawashima, M. & Ishiguro-Watanabe, M. KEGG for taxonomy-based analysis of pathways and genomes. Nucleic Acids Research vol. 51 D587–D592 (2022).",
  "10.1093/nar/gkac972": " Groza, T. et al. The International Mouse Phenotyping Consortium: comprehensive knockout phenotyping underpinning the study of human disease. Nucleic Acids Research vol. 51 D1038–D1045 (2022).",
  "10.1093/nar/gkad960": " Agrawal, A. et al. WikiPathways 2024: next generation pathway database. Nucleic Acids Research vol. 52 D679–D689 (2023).",
  "10.1093/nar/gks1114": "Wu, C., MacLeod, I. & Su, A. I. BioGPS and MyGene.info: organizing online, gene-centric information. Nucleic Acids Research vol. 41 D561–D565 (2012).",
  "10.1093/nar/gks1193": "Barrett, T. et al. NCBI GEO: archive for functional genomics data sets—update. Nucleic Acids Research vol. 41 D991–D995 (2012).",
  "10.1093/nar/gku1003": " Szklarczyk, D. et al. STRING v10: protein–protein interaction networks, integrated over the tree of life. Nucleic Acids Research vol. 43 D447–D452 (2014).",
  "10.1093/nar/gkv007": " Ritchie, M. E. et al. limma powers differential expression analyses for RNA-sequencing and microarray studies. Nucleic Acids Research vol. 43 e47–e47 (2015).",
  "10.1093/nar/gkx1153": "Landrum, M. J. et al. ClinVar: improving access to variant interpretations and supporting evidence. Nucleic Acids Research vol. 46 D1062–D1067 (2017).",
  "10.1093/nar/gkz393": " Lachmann, A. et al. Geneshot: search engine for ranking genes from arbitrary text queries. Nucleic Acids Research vol. 47 W571–W577 (2019).",
  "10.1093/nar/gkz446": "Keenan, A. B. et al. ChEA3: transcription factor enrichment analysis by orthogonal omics integration. Nucleic Acids Research vol. 47 W212–W224 (2019).",
  "10.1186/1471-2105-15-79": " Clark, N. R. et al. The characteristic direction: a geometrical approach to identify differentially expressed genes. BMC Bioinformatics vol. 15 (2014).",
  "10.1186/gb-2014-15-2-r29": " Law, C. W., Chen, Y., Shi, W. & Smyth, G. K. voom: precision weights unlock linear model analysis tools for RNA-seq read counts. Genome Biology vol. 15 R29 (2014).",
  "10.1186/s12859-022-04590-5": " Kropiwnicki, E. et al. DrugShot: querying biomedical search terms to retrieve prioritized lists of small molecules. BMC Bioinformatics vol. 23 (2022).",
  "10.1186/s13059-016-0953-9": "Xin, J. et al. High-performance web services for querying gene and variant annotation. Genome Biology vol. 17 (2016).",
  "10.1371/journal.pcbi.1009551": " Thistlethwaite, L. R. et al. Correction: CTD: An information-theoretic algorithm to interpret sets of metabolomic and transcriptomic perturbations in the context of graphical models. PLOS Computational Biology vol. 17 e1009551 (2021).",
  "10.1371/journal.pmed.1001779": "Sudlow, C. et al. UK Biobank: An Open Access Resource for Identifying the Causes of a Wide Range of Complex Diseases of Middle and Old Age. PLOS Medicine vol. 12 e1001779 (2015).",
  "10.48550/arXiv.1802.03426": " McInnes, L., Healy, J. & Melville, J. UMAP: Uniform Manifold Approximation and Projection for Dimension Reduction. Preprint at https://doi.org/10.48550/ARXIV.1802.03426 (2018).",
}