export default {
  "10.1002/cpz1.90": {"nature":"Xie, Z. et al. Gene Set Knowledge Discovery with Enrichr. Current Protocols vol. 1 (2021).","bibtex":{"type":"misc","record":" title={Gene Set Knowledge Discovery with Enrichr}, volume={1}, url={http://dx.doi.org/10.1002/cpz1.90}, DOI={10.1002/cpz1.90}, abstractNote={<jats:title>Abstract</jats:title><jats:p>Profiling samples from patients, tissues, and cells with genomics, transcriptomics, epigenomics, proteomics, and metabolomics ultimately produces lists of genes and proteins that need to be further analyzed and integrated in the context of known biology. Enrichr (Chen et al., 2013; Kuleshov et al., 2016) is a gene set search engine that enables the querying of hundreds of thousands of annotated gene sets. Enrichr uniquely integrates knowledge from many high‐profile projects to provide synthesized information about mammalian genes and gene sets. The platform provides various methods to compute gene set enrichment, and the results are visualized in several interactive ways. This protocol provides a summary of the key features of Enrichr, which include using Enrichr programmatically and embedding an Enrichr button on any website. © 2021 Wiley Periodicals LLC.</jats:p><jats:p><jats:bold>Basic Protocol 1</jats:bold>: Analyzing lists of differentially expressed genes from transcriptomics, proteomics and phosphoproteomics, GWAS studies, or other experimental studies</jats:p><jats:p><jats:bold>Basic Protocol 2</jats:bold>: Searching Enrichr by a single gene or key search term</jats:p><jats:p><jats:bold>Basic Protocol 3</jats:bold>: Preparing raw or processed RNA‐seq data through BioJupies in preparation for Enrichr analysis</jats:p><jats:p><jats:bold>Basic Protocol 4</jats:bold>: Analyzing gene sets for model organisms using modEnrichr</jats:p><jats:p><jats:bold>Basic Protocol 5</jats:bold>: Using Enrichr in Geneshot</jats:p><jats:p><jats:bold>Basic Protocol 6</jats:bold>: Using Enrichr in ARCHS4</jats:p><jats:p><jats:bold>Basic Protocol 7</jats:bold>: Using the enrichment analysis visualization Appyter to visualize Enrichr results</jats:p><jats:p><jats:bold>Basic Protocol 8</jats:bold>: Using the Enrichr API</jats:p><jats:p><jats:bold>Basic Protocol 9</jats:bold>: Adding an Enrichr button to a website</jats:p>}, number={3}, journal={Current Protocols}, publisher={Wiley}, author={Xie, Zhuorui and Bailey, Allison and Kuleshov, Maxim V. and Clarke, Daniel J. B. and Evangelista, John E. and Jenkins, Sherry L. and Lachmann, Alexander and Wojciechowicz, Megan L. and Kropiwnicki, Eryk and Jagodnik, Kathleen M. and Jeon, Minji and Ma’ayan, Avi}, year={2021}, month=mar, language={en} "}},
  "10.1002/pro.3715": {"nature":"Kanehisa, M. Toward understanding the origin and evolution of cellular organisms. Protein Science vol. 28 1947–1951 (2019).","bibtex":{"type":"misc","record":" title={Toward understanding the origin and evolution of cellular organisms}, volume={28}, url={http://dx.doi.org/10.1002/pro.3715}, DOI={10.1002/pro.3715}, abstractNote={<jats:title>Abstract</jats:title><jats:p>In this era of high‐throughput biology, bioinformatics has become a major discipline for making sense out of large‐scale datasets. Bioinformatics is usually considered as a practical field developing databases and software tools for supporting other fields, rather than a fundamental scientific discipline for uncovering principles of biology. The KEGG resource that we have been developing is a reference knowledge base for biological interpretation of genome sequences and other high‐throughput data. It is now one of the most utilized biological databases because of its practical values. For me personally, KEGG is a step toward understanding the origin and evolution of cellular organisms.</jats:p>}, number={11}, journal={Protein Science}, publisher={Wiley}, author={Kanehisa, Minoru}, year={2019}, month=sep, pages={1947–1951}, language={en} "}},
  "10.1016/j.cell.2017.10.049": {"nature":"Subramanian, A. et al. A Next Generation Connectivity Map: L1000 Platform and the First 1,000,000 Profiles. Cell vol. 171 1437-1452.e17 (2017).","bibtex":{"type":"misc","record":" title={A Next Generation Connectivity Map: L1000 Platform and the First 1,000,000 Profiles}, volume={171}, url={http://dx.doi.org/10.1016/j.cell.2017.10.049}, DOI={10.1016/j.cell.2017.10.049}, number={6}, journal={Cell}, publisher={Elsevier BV}, author={Subramanian, Aravind and Narayan, Rajiv and Corsello, Steven M. and Peck, David D. and Natoli, Ted E. and Lu, Xiaodong and Gould, Joshua and Davis, John F. and Tubelli, Andrew A. and Asiedu, Jacob K. and Lahr, David L. and Hirschman, Jodi E. and Liu, Zihan and Donahue, Melanie and Julian, Bina and Khan, Mariya and Wadden, David and Smith, Ian C. and Lam, Daniel and Liberzon, Arthur and Toder, Courtney and Bagul, Mukta and Orzechowski, Marek and Enache, Oana M. and Piccioni, Federica and Johnson, Sarah A. and Lyons, Nicholas J. and Berger, Alice H. and Shamji, Alykhan F. and Brooks, Angela N. and Vrcic, Anita and Flynn, Corey and Rosains, Jacqueline and Takeda, David Y. and Hu, Roger and Davison, Desiree and Lamb, Justin and Ardlie, Kristin and Hogstrom, Larson and Greenside, Peyton and Gray, Nathanael S. and Clemons, Paul A. and Silver, Serena and Wu, Xiaoyun and Zhao, Wen-Ning and Read-Button, Willis and Wu, Xiaohua and Haggarty, Stephen J. and Ronco, Lucienne V. and Boehm, Jesse S. and Schreiber, Stuart L. and Doench, John G. and Bittker, Joshua A. and Root, David E. and Wong, Bang and Golub, Todd R.}, year={2017}, month=nov, pages={1437-1452.e17}, language={en} "}},
  "10.1016/j.cell.2020.06.004": {"nature":"Sanford, J. A. et al. Molecular Transducers of Physical Activity Consortium (MoTrPAC): Mapping the Dynamic Responses to Exercise. Cell vol. 181 1464–1474 (2020).","bibtex":{"type":"misc","record":" title={Molecular Transducers of Physical Activity Consortium (MoTrPAC): Mapping the Dynamic Responses to Exercise}, volume={181}, url={http://dx.doi.org/10.1016/j.cell.2020.06.004}, DOI={10.1016/j.cell.2020.06.004}, number={7}, journal={Cell}, publisher={Elsevier BV}, author={Sanford, James A. and Nogiec, Christopher D. and Lindholm, Malene E. and Adkins, Joshua N. and Amar, David and Dasari, Surendra and Drugan, Jonelle K. and Fernández, Facundo M. and Radom-Aizik, Shlomit and Schenk, Simon and Snyder, Michael P. and Tracy, Russell P. and Vanderboom, Patrick and Trappe, Scott and Walsh, Martin J. and Adkins, Joshua N. and Amar, David and Dasari, Surendra and Drugan, Jonelle K. and Evans, Charles R. and Fernandez, Facundo M. and Li, Yafeng and Lindholm, Malene E. and Nogiec, Christopher D. and Radom-Aizik, Shlomit and Sanford, James A. and Schenk, Simon and Snyder, Michael P. and Tomlinson, Lyl and Tracy, Russell P. and Trappe, Scott and Vanderboom, Patrick and Walsh, Martin J. and Lee Alekel, D. and Bekirov, Iddil and Boyce, Amanda T. and Boyington, Josephine and Fleg, Jerome L. and Joseph, Lyndon J.O. and Laughlin, Maren R. and Maruvada, Padma and Morris, Stephanie A. and McGowan, Joan A. and Nierras, Concepcion and Pai, Vinay and Peterson, Charlotte and Ramos, Ed and Roary, Mary C. and Williams, John P. and Xia, Ashley and Cornell, Elaine and Rooney, Jessica and Miller, Michael E. and Ambrosius, Walter T. and Rushing, Scott and Stowe, Cynthia L. and Jack Rejeski, W. and Nicklas, Barbara J. and Pahor, Marco and Lu, Ching-ju and Trappe, Todd and Chambers, Toby and Raue, Ulrika and Lester, Bridget and Bergman, Bryan C. and Bessesen, David H. and Jankowski, Catherine M. and Kohrt, Wendy M. and Melanson, Edward L. and Moreau, Kerrie L. and Schauer, Irene E. and Schwartz, Robert S. and Kraus, William E. and Slentz, Cris A. and Huffman, Kim M. and Johnson, Johanna L. and Willis, Leslie H. and Kelly, Leslie and Houmard, Joseph A. and Dubis, Gabriel and Broskey, Nick and Goodpaster, Bret H. and Sparks, Lauren M. and Coen, Paul M. and Cooper, Dan M. and Haddad, Fadia and Rankinen, Tuomo and Ravussin, Eric and Johannsen, Neil and Harris, Melissa and Jakicic, John M. and Newman, Anne B. and Forman, Daniel D. and Kershaw, Erin and Rogers, Renee J. and Nindl, Bradley C. and Page, Lindsay C. and Stefanovic-Racic, Maja and Barr, Susan L. and Rasmussen, Blake B. and Moro, Tatiana and Paddon-Jones, Doug and Volpi, Elena and Spratt, Heidi and Musi, Nicolas and Espinoza, Sara and Patel, Darpan and Serra, Monica and Gelfond, Jonathan and Burns, Aisling and Bamman, Marcas M. and Buford, Thomas W. and Cutter, Gary R. and Bodine, Sue C. and Esser, Karyn and Farrar, Rodger P. and Goodyear, Laurie J. and Hirshman, Michael F. and Albertson, Brent G. and Qian, Wei-Jun and Piehowski, Paul and Gritsenko, Marina A. and Monore, Matthew E. and Petyuk, Vladislav A. and McDermott, Jason E. and Hansen, Joshua N. and Hutchison, Chelsea and Moore, Samuel and Gaul, David A. and Clish, Clary B. and Avila-Pacheco, Julian and Dennis, Courtney and Kellis, Manolis and Carr, Steve and Jean-Beltran, Pierre M. and Keshishian, Hasmik and Mani, D.R. and Clauser, Karl and Krug, Karsten and Mundorff, Charlie and Pearce, Cadence and Ivanova, Anna A. and Ortlund, Eric A. and Maner-Smith, Kristal and Uppal, Karan and Zhang, Tiantian and Sealfon, Stuart C. and Zaslavsky, Elena and Nair, Venugopalan and Li, SiDe and Jain, Nimisha and Ge, YongChao and Sun, Yifei and Nudelman, German and Ruf-zamojski, Frederique and Smith, Gregory and Pincas, Nhanna and Rubenstein, Aliza and Anne Amper, Mary and Seenarine, Nitish and Lappalainen, Tuuli and Lanza, Ian R. and Sreekumaran Nair, K. and Klaus, Katherine and Montgomery, Stephen B. and Smith, Kevin S. and Gay, Nicole R. and Zhao, Bingqing and Hung, Chia-Jiu and Zebarjadi, Navid and Balliu, Brunilda and Fresard, Laure and Burant, Charles F. and Li, Jun Z. and Kachman, Maureen and Soni, Tanu and Raskind, Alexander B. and Gerszten, Robert and Robbins, Jeremy and Ilkayeva, Olga and Muehlbauer, Michael J. and Newgard, Christopher B. and Ashley, Euan A. and Wheeler, Matthew T. and Jimenez-Morales, David and Raja, Archana and Dalton, Karen P. and Zhen, Jimmy and Suk Kim, Young and Christle, Jeffrey W. and Marwaha, Shruti and Chin, Elizabeth T. and Hershman, Steven G. and Hastie, Trevor and Tibshirani, Robert and Rivas, Manuel A.}, year={2020}, month=jun, pages={1464–1474}, language={en} "}},
  "10.1016/j.isci.2022.105799": {"nature":"Petrosyan, V. et al. Identifying biomarkers of differential chemotherapy response in TNBC patient-derived xenografts with a CTD/WGCNA approach. iScience vol. 26 105799 (2023).","bibtex":{"type":"misc","record":" title={Identifying biomarkers of differential chemotherapy response in TNBC patient-derived xenografts with a CTD/WGCNA approach}, volume={26}, url={http://dx.doi.org/10.1016/j.isci.2022.105799}, DOI={10.1016/j.isci.2022.105799}, number={1}, journal={iScience}, publisher={Elsevier BV}, author={Petrosyan, Varduhi and Dobrolecki, Lacey E. and Thistlethwaite, Lillian and Lewis, Alaina N. and Sallas, Christina and Srinivasan, Ramakrishnan R. and Lei, Jonathan T. and Kovacevic, Vladimir and Obradovic, Predrag and Ellis, Matthew J. and Osborne, C. Kent and Rimawi, Mothaffar F. and Pavlick, Anne and Shafaee, Maryam Nemati and Dowst, Heidi and Jain, Antrix and Saltzman, Alexander B. and Malovannaya, Anna and Marangoni, Elisabetta and Welm, Alana L. and Welm, Bryan E. and Li, Shunqiang and Wulf, Gerburg M. and Sonzogni, Olmo and Huang, Chen and Vasaikar, Suhas and Hilsenbeck, Susan G. and Zhang, Bing and Milosavljevic, Aleksandar and Lewis, Michael T.}, year={2023}, month=jan, pages={105799}, language={en} "}},
  "10.1016/j.xgen.2023.100340": {"nature":"Shapiro, J. A. et al. OpenPBTA: The Open Pediatric Brain Tumor Atlas. Cell Genomics vol. 3 100340 (2023).","bibtex":{"type":"misc","record":" title={OpenPBTA: The Open Pediatric Brain Tumor Atlas}, volume={3}, url={http://dx.doi.org/10.1016/j.xgen.2023.100340}, DOI={10.1016/j.xgen.2023.100340}, number={7}, journal={Cell Genomics}, publisher={Elsevier BV}, author={Shapiro, Joshua A. and Gaonkar, Krutika S. and Spielman, Stephanie J. and Savonen, Candace L. and Bethell, Chante J. and Jin, Run and Rathi, Komal S. and Zhu, Yuankun and Egolf, Laura E. and Farrow, Bailey K. and Miller, Daniel P. and Yang, Yang and Koganti, Tejaswi and Noureen, Nighat and Koptyra, Mateusz P. and Duong, Nhat and Santi, Mariarita and Kim, Jung and Robins, Shannon and Storm, Phillip B. and Mack, Stephen C. and Lilly, Jena V. and Xie, Hongbo M. and Jain, Payal and Raman, Pichai and Rood, Brian R. and Lulla, Rishi R. and Nazarian, Javad and Kraya, Adam A. and Vaksman, Zalman and Heath, Allison P. and Kline, Cassie and Scolaro, Laura and Viaene, Angela N. and Huang, Xiaoyan and Way, Gregory P. and Foltz, Steven M. and Zhang, Bo and Poetsch, Anna R. and Mueller, Sabine and Ennis, Brian M. and Prados, Michael and Diskin, Sharon J. and Zheng, Siyuan and Guo, Yiran and Kannan, Shrivats and Waanders, Angela J. and Margol, Ashley S. and Kim, Meen Chul and Hanson, Derek and Van Kuren, Nicholas and Wong, Jessica and Kaufman, Rebecca S. and Coleman, Noel and Blackden, Christopher and Cole, Kristina A. and Mason, Jennifer L. and Madsen, Peter J. and Koschmann, Carl J. and Stewart, Douglas R. and Wafula, Eric and Brown, Miguel A. and Resnick, Adam C. and Greene, Casey S. and Rokita, Jo Lynne and Taroni, Jaclyn N.}, year={2023}, month=jul, pages={100340}, language={en} "}},
  "10.1017/CBO9781139058452.002": {"nature":"Data Mining. Mining of Massive Datasets 1–17 (2011) doi:10.1017/cbo9781139058452.002.","bibtex":{"type":"misc","record":" url={http://dx.doi.org/10.1017/CBO9781139058452.002}, DOI={10.1017/cbo9781139058452.002}, journal={Mining of Massive Datasets}, publisher={Cambridge University Press}, year={2011}, month=oct, pages={1–17} "}},
  "10.1038/75556": {"nature":"Ashburner, M. et al. Gene Ontology: tool for the unification of biology. Nature Genetics vol. 25 25–29 (2000).","bibtex":{"type":"misc","record":" title={Gene Ontology: tool for the unification of biology}, volume={25}, url={http://dx.doi.org/10.1038/75556}, DOI={10.1038/75556}, number={1}, journal={Nature Genetics}, publisher={Springer Science and Business Media LLC}, author={Ashburner, Michael and Ball, Catherine A. and Blake, Judith A. and Botstein, David and Butler, Heather and Cherry, J. Michael and Davis, Allan P. and Dolinski, Kara and Dwight, Selina S. and Eppig, Janan T. and Harris, Midori A. and Hill, David P. and Issel-Tarver, Laurie and Kasarskis, Andrew and Lewis, Suzanna and Matese, John C. and Richardson, Joel E. and Ringwald, Martin and Rubin, Gerald M. and Sherlock, Gavin}, year={2000}, month=may, pages={25–29}, language={en} "}},
  "10.1038/nature11247": {"nature":"An integrated encyclopedia of DNA elements in the human genome. Nature vol. 489 57–74 (2012).","bibtex":{"type":"misc","record":" title={An integrated encyclopedia of DNA elements in the human genome}, volume={489}, url={http://dx.doi.org/10.1038/nature11247}, DOI={10.1038/nature11247}, number={7414}, journal={Nature}, publisher={Springer Science and Business Media LLC}, year={2012}, month=sep, pages={57–74}, language={en} "}},
  "10.1038/ng.265": {"nature":"Pannicke, U. et al. Reticular dysgenesis (aleukocytosis) is caused by mutations in the gene encoding mitochondrial adenylate kinase 2. Nature Genetics vol. 41 101–105 (2008).","bibtex":{"type":"misc","record":" title={Reticular dysgenesis (aleukocytosis) is caused by mutations in the gene encoding mitochondrial adenylate kinase 2}, volume={41}, url={http://dx.doi.org/10.1038/ng.265}, DOI={10.1038/ng.265}, number={1}, journal={Nature Genetics}, publisher={Springer Science and Business Media LLC}, author={Pannicke, Ulrich and Hönig, Manfred and Hess, Isabell and Friesen, Claudia and Holzmann, Karlheinz and Rump, Eva-Maria and Barth, Thomas F and Rojewski, Markus T and Schulz, Ansgar and Boehm, Thomas and Friedrich, Wilhelm and Schwarz, Klaus}, year={2008}, month=nov, pages={101–105}, language={en} "}},
  "10.1038/ng.2653": {"nature":"Lonsdale, J. et al. The Genotype-Tissue Expression (GTEx) project. Nature Genetics vol. 45 580–585 (2013).","bibtex":{"type":"misc","record":" title={The Genotype-Tissue Expression (GTEx) project}, volume={45}, url={http://dx.doi.org/10.1038/ng.2653}, DOI={10.1038/ng.2653}, number={6}, journal={Nature Genetics}, publisher={Springer Science and Business Media LLC}, author={Lonsdale, John and Thomas, Jeffrey and Salvatore, Mike and Phillips, Rebecca and Lo, Edmund and Shad, Saboor and Hasz, Richard and Walters, Gary and Garcia, Fernando and Young, Nancy and Foster, Barbara and Moser, Mike and Karasik, Ellen and Gillard, Bryan and Ramsey, Kimberley and Sullivan, Susan and Bridge, Jason and Magazine, Harold and Syron, John and Fleming, Johnelle and Siminoff, Laura and Traino, Heather and Mosavel, Maghboeba and Barker, Laura and Jewell, Scott and Rohrer, Dan and Maxim, Dan and Filkins, Dana and Harbach, Philip and Cortadillo, Eddie and Berghuis, Bree and Turner, Lisa and Hudson, Eric and Feenstra, Kristin and Sobin, Leslie and Robb, James and Branton, Phillip and Korzeniewski, Greg and Shive, Charles and Tabor, David and Qi, Liqun and Groch, Kevin and Nampally, Sreenath and Buia, Steve and Zimmerman, Angela and Smith, Anna and Burges, Robin and Robinson, Karna and Valentino, Kim and Bradbury, Deborah and Cosentino, Mark and Diaz-Mayoral, Norma and Kennedy, Mary and Engel, Theresa and Williams, Penelope and Erickson, Kenyon and Ardlie, Kristin and Winckler, Wendy and Getz, Gad and DeLuca, David and MacArthur, Daniel and Kellis, Manolis and Thomson, Alexander and Young, Taylor and Gelfand, Ellen and Donovan, Molly and Meng, Yan and Grant, George and Mash, Deborah and Marcus, Yvonne and Basile, Margaret and Liu, Jun and Zhu, Jun and Tu, Zhidong and Cox, Nancy J and Nicolae, Dan L and Gamazon, Eric R and Im, Hae Kyung and Konkashbaev, Anuar and Pritchard, Jonathan and Stevens, Matthew and Flutre, Timothèe and Wen, Xiaoquan and Dermitzakis, Emmanouil T and Lappalainen, Tuuli and Guigo, Roderic and Monlong, Jean and Sammeth, Michael and Koller, Daphne and Battle, Alexis and Mostafavi, Sara and McCarthy, Mark and Rivas, Manual and Maller, Julian and Rusyn, Ivan and Nobel, Andrew and Wright, Fred and Shabalin, Andrey and Feolo, Mike and Sharopova, Nataliya and Sturcke, Anne and Paschal, Justin and Anderson, James M and Wilder, Elizabeth L and Derr, Leslie K and Green, Eric D and Struewing, Jeffery P and Temple, Gary and Volpi, Simona and Boyer, Joy T and Thomson, Elizabeth J and Guyer, Mark S and Ng, Cathy and Abdallah, Assya and Colantuoni, Deborah and Insel, Thomas R and Koester, Susan E and Little, A Roger and Bender, Patrick K and Lehner, Thomas and Yao, Yin and Compton, Carolyn C and Vaught, Jimmie B and Sawyer, Sherilyn and Lockhart, Nicole C and Demchok, Joanne and Moore, Helen F}, year={2013}, month=may, pages={580–585}, language={en} "}},
  "10.1038/s41467-018-03751-6": {"nature":"Lachmann, A. et al. Massive mining of publicly available RNA-seq data from human and mouse. Nature Communications vol. 9 (2018).","bibtex":{"type":"misc","record":" title={Massive mining of publicly available RNA-seq data from human and mouse}, volume={9}, url={http://dx.doi.org/10.1038/s41467-018-03751-6}, DOI={10.1038/s41467-018-03751-6}, abstractNote={<jats:title>Abstract</jats:title><jats:p>RNA sequencing (RNA-seq) is the leading technology for genome-wide transcript quantification. However, publicly available RNA-seq data is currently provided mostly in raw form, a significant barrier for global and integrative retrospective analyses. ARCHS4 is a web resource that makes the majority of published RNA-seq data from human and mouse available at the gene and transcript levels. For developing ARCHS4, available FASTQ files from RNA-seq experiments from the Gene Expression Omnibus (GEO) were aligned using a cloud-based infrastructure. In total 187,946 samples are accessible through ARCHS4 with 103,083 mouse and 84,863 human. Additionally, the ARCHS4 web interface provides intuitive exploration of the processed data through querying tools, interactive visualization, and gene pages that provide average expression across cell lines and tissues, top co-expressed genes for each gene, and predicted biological functions and protein–protein interactions for each gene based on prior knowledge combined with co-expression.</jats:p>}, number={1}, journal={Nature Communications}, publisher={Springer Science and Business Media LLC}, author={Lachmann, Alexander and Torre, Denis and Keenan, Alexandra B. and Jagodnik, Kathleen M. and Lee, Hoyjin J. and Wang, Lily and Silverstein, Moshe C. and Ma’ayan, Avi}, year={2018}, month=apr, language={en} "}},
  "10.1038/s41598-020-72664-6": {"nature":"Zhao, Y., Wong, L. & Goh, W. W. B. How to do quantile normalization correctly for gene expression data analyses. Scientific Reports vol. 10 (2020).","bibtex":{"type":"misc","record":" title={How to do quantile normalization correctly for gene expression data analyses}, volume={10}, url={http://dx.doi.org/10.1038/s41598-020-72664-6}, DOI={10.1038/s41598-020-72664-6}, abstractNote={<jats:title>Abstract</jats:title><jats:p>Quantile normalization is an important normalization technique commonly used in high-dimensional data analysis. However, it is susceptible to class-effect proportion effects (the proportion of class-correlated variables in a dataset) and batch effects (the presence of potentially confounding technical variation) when applied blindly on whole data sets, resulting in higher false-positive and false-negative rates. We evaluate five strategies for performing quantile normalization, and demonstrate that good performance in terms of batch-effect correction and statistical feature selection can be readily achieved by first splitting data by sample class-labels before performing quantile normalization independently on each split (“Class-specific”). Via simulations with both real and simulated batch effects, we demonstrate that the “Class-specific” strategy (and others relying on similar principles) readily outperform whole-data quantile normalization, and is robust-preserving useful signals even during the combined analysis of separately-normalized datasets. Quantile normalization is a commonly used procedure. But when carelessly applied on whole datasets without first considering class-effect proportion and batch effects, can result in poor performance. If quantile normalization must be used, then we recommend using the “Class-specific” strategy.</jats:p>}, number={1}, journal={Scientific Reports}, publisher={Springer Science and Business Media LLC}, author={Zhao, Yaxing and Wong, Limsoon and Goh, Wilson Wen Bin}, year={2020}, month=sep, language={en} "}},
  "10.1073/pnas.0506580102": {"nature":"Subramanian, A. et al. Gene set enrichment analysis: A knowledge-based approach for interpreting genome-wide expression profiles. Proceedings of the National Academy of Sciences vol. 102 15545–15550 (2005).","bibtex":{"type":"misc","record":" title={Gene set enrichment analysis: A knowledge-based approach for interpreting genome-wide expression profiles}, volume={102}, url={http://dx.doi.org/10.1073/pnas.0506580102}, DOI={10.1073/pnas.0506580102}, abstractNote={<jats:p>Although genomewide RNA expression analysis has become a routine tool in biomedical research, extracting biological insight from such information remains a major challenge. Here, we describe a powerful analytical method called Gene Set Enrichment Analysis (GSEA) for interpreting gene expression data. The method derives its power by focusing on gene sets, that is, groups of genes that share common biological function, chromosomal location, or regulation. We demonstrate how GSEA yields insights into several cancer-related data sets, including leukemia and lung cancer. Notably, where single-gene analysis finds little similarity between two independent studies of patient survival in lung cancer, GSEA reveals many biological pathways in common. The GSEA method is embodied in a freely available software package, together with an initial database of 1,325 biologically defined gene sets.</jats:p>}, number={43}, journal={Proceedings of the National Academy of Sciences}, publisher={Proceedings of the National Academy of Sciences}, author={Subramanian, Aravind and Tamayo, Pablo and Mootha, Vamsi K. and Mukherjee, Sayan and Ebert, Benjamin L. and Gillette, Michael A. and Paulovich, Amanda and Pomeroy, Scott L. and Golub, Todd R. and Lander, Eric S. and Mesirov, Jill P.}, year={2005}, month=sep, pages={15545–15550}, language={en} "}},
  "10.1093/bioinformatics/btac017": {"nature":"Lelong, S. et al. BioThings SDK: a toolkit for building high-performance data APIs in biomedical research. Bioinformatics vol. 38 2077–2079 (2022).","bibtex":{"type":"misc","record":" title={BioThings SDK: a toolkit for building high-performance data APIs in biomedical research}, volume={38}, url={http://dx.doi.org/10.1093/bioinformatics/btac017}, DOI={10.1093/bioinformatics/btac017}, abstractNote={<jats:title>Abstract</jats:title>\n               <jats:sec>\n                  <jats:title>Summary</jats:title>\n                  <jats:p>To meet the increased need of making biomedical resources more accessible and reusable, Web Application Programming Interfaces (APIs) or web services have become a common way to disseminate knowledge sources. The BioThings APIs are a collection of high-performance, scalable, annotation as a service APIs that automate the integration of biological annotations from disparate data sources. This collection of APIs currently includes MyGene.info, MyVariant.info and MyChem.info for integrating annotations on genes, variants and chemical compounds, respectively. These APIs are used by both individual researchers and application developers to simplify the process of annotation retrieval and identifier mapping. Here, we describe the BioThings Software Development Kit (SDK), a generalizable and reusable toolkit for integrating data from multiple disparate data sources and creating high-performance APIs. This toolkit allows users to easily create their own BioThings APIs for any data type of interest to them, as well as keep APIs up-to-date with their underlying data sources.</jats:p>\n               </jats:sec>\n               <jats:sec>\n                  <jats:title>Availability and implementation</jats:title>\n                  <jats:p>The BioThings SDK is built in Python and released via PyPI (https://pypi.org/project/biothings/). Its source code is hosted at its github repository (https://github.com/biothings/biothings.api).</jats:p>\n               </jats:sec>\n               <jats:sec>\n                  <jats:title>Supplementary information</jats:title>\n                  <jats:p>Supplementary data are available at Bioinformatics online.</jats:p>\n               </jats:sec>}, number={7}, journal={Bioinformatics}, publisher={Oxford University Press (OUP)}, author={Lelong, Sebastien and Zhou, Xinghua and Afrasiabi, Cyrus and Qian, Zhongchao and Cano, Marco Alvarado and Tsueng, Ginger and Xin, Jiwen and Mullen, Julia and Yao, Yao and Avila, Ricardo and Taylor, Greg and Su, Andrew I and Wu, Chunlei}, editor={Lu, Zhiyong}, year={2022}, month=jan, pages={2077–2079}, language={en} "}},
  "10.1093/database/baab017": {"nature":"Kropiwnicki, E. et al. Drugmonizome and Drugmonizome-ML: integration and abstraction of small molecule attributes for drug enrichment analysis and machine learning. Database vol. 2021 (2021).","bibtex":{"type":"misc","record":" title={Drugmonizome and Drugmonizome-ML: integration and abstraction of small molecule attributes for drug enrichment analysis and machine learning}, volume={2021}, url={http://dx.doi.org/10.1093/database/baab017}, DOI={10.1093/database/baab017}, abstractNote={<jats:title>Abstract</jats:title><jats:p>Understanding the underlying molecular and structural similarities between seemingly heterogeneous sets of drugs can aid in identifying drug repurposing opportunities and assist in the discovery of novel properties of preclinical small molecules. A wealth of information about drug and small molecule structure, targets, indications and side effects; induced gene expression signatures; and other attributes are publicly available through web-based tools, databases and repositories. By processing, abstracting and aggregating information from these resources into drug set libraries, knowledge about novel properties of drugs and small molecules can be systematically imputed with machine learning. In addition, drug set libraries can be used as the underlying database for drug set enrichment analysis. Here, we present Drugmonizome, a database with a search engine for querying annotated sets of drugs and small molecules for performing drug set enrichment analysis. Utilizing the data within Drugmonizome, we also developed Drugmonizome-ML. Drugmonizome-ML enables users to construct customized machine learning pipelines using the drug set libraries from Drugmonizome. To demonstrate the utility of Drugmonizome, drug sets from 12 independent SARS-CoV-2 in vitro screens were subjected to consensus enrichment analysis. Despite the low overlap among these 12 independent in vitro screens, we identified common biological processes critical for blocking viral replication. To demonstrate Drugmonizome-ML, we constructed a machine learning pipeline to predict whether approved and preclinical drugs may induce peripheral neuropathy as a potential side effect. Overall, the Drugmonizome and Drugmonizome-ML resources provide rich and diverse knowledge about drugs and small molecules for direct systems pharmacology applications.</jats:p><jats:p>Database URL: https://maayanlab.cloud/drugmonizome/.</jats:p>}, journal={Database}, publisher={Oxford University Press (OUP)}, author={Kropiwnicki, Eryk and Evangelista, John E and Stein, Daniel J and Clarke, Daniel J B and Lachmann, Alexander and Kuleshov, Maxim V and Jeon, Minji and Jagodnik, Kathleen M and Ma’ayan, Avi}, year={2021}, month=jan, language={en} "}},
  "10.1093/glycob/cwx066": {"nature":"Tiemeyer, M. et al. GlyTouCan: an accessible glycan structure repository. Glycobiology vol. 27 915–919 (2017).","bibtex":{"type":"misc","record":" title={GlyTouCan: an accessible glycan structure repository}, volume={27}, url={http://dx.doi.org/10.1093/glycob/cwx066}, DOI={10.1093/glycob/cwx066}, number={10}, journal={Glycobiology}, publisher={Oxford University Press (OUP)}, author={Tiemeyer, Michael and Aoki, Kazuhiro and Paulson, James and Cummings, Richard D and York, William S and Karlsson, Niclas G and Lisacek, Frederique and Packer, Nicolle H and Campbell, Matthew P and Aoki, Nobuyuki P and Fujita, Akihiro and Matsubara, Masaaki and Shinmachi, Daisuke and Tsuchiya, Shinichiro and Yamada, Issaku and Pierce, Michael and Ranzinger, René and Narimatsu, Hisashi and Aoki-Kinoshita, Kiyoko F}, year={2017}, month=aug, pages={915–919}, language={en} "}},
  "10.1093/glycob/cwz080": {"nature":"York, W. S. et al. GlyGen: Computational and Informatics Resources for Glycoscience. Glycobiology vol. 30 72–73 (2019).","bibtex":{"type":"misc","record":" title={GlyGen: Computational and Informatics Resources for Glycoscience}, volume={30}, url={http://dx.doi.org/10.1093/glycob/cwz080}, DOI={10.1093/glycob/cwz080}, number={2}, journal={Glycobiology}, publisher={Oxford University Press (OUP)}, author={York, William S and Mazumder, Raja and Ranzinger, Rene and Edwards, Nathan and Kahsay, Robel and Aoki-Kinoshita, Kiyoko F and Campbell, Matthew P and Cummings, Richard D and Feizi, Ten and Martin, Maria and Natale, Darren A and Packer, Nicolle H and Woods, Robert J and Agarwal, Gaurav and Arpinar, Sena and Bhat, Sanath and Blake, Judith and Castro, Leyla Jael Garcia and Fochtman, Brian and Gildersleeve, Jeffrey and Goldman, Radoslav and Holmes, Xavier and Jain, Vinamra and Kulkarni, Sujeet and Mahadik, Rupali and Mehta, Akul and Mousavi, Reza and Nakarakommula, Sandeep and Navelkar, Rahi and Pattabiraman, Nagarajan and Pierce, Michael J and Ross, Karen and Vasudev, Preethi and Vora, Jeet and Williamson, Tatiana and Zhang, Wenjin}, year={2019}, month=oct, pages={72–73}, language={en} "}},
  "10.1093/nar/gkaa1024": {"nature":"Martens, M. et al. WikiPathways: connecting communities. Nucleic Acids Research vol. 49 D613–D621 (2020).","bibtex":{"type":"misc","record":" title={WikiPathways: connecting communities}, volume={49}, url={http://dx.doi.org/10.1093/nar/gkaa1024}, DOI={10.1093/nar/gkaa1024}, abstractNote={<jats:title>Abstract</jats:title><jats:p>WikiPathways (https://www.wikipathways.org) is a biological pathway database known for its collaborative nature and open science approaches. With the core idea of the scientific community developing and curating biological knowledge in pathway models, WikiPathways lowers all barriers for accessing and using its content. Increasingly more content creators, initiatives, projects and tools have started using WikiPathways. Central in this growth and increased use of WikiPathways are the various communities that focus on particular subsets of molecular pathways such as for rare diseases and lipid metabolism. Knowledge from published pathway figures helps prioritize pathway development, using optical character and named entity recognition. We show the growth of WikiPathways over the last three years, highlight the new communities and collaborations of pathway authors and curators, and describe various technologies to connect to external resources and initiatives. The road toward a sustainable, community-driven pathway database goes through integration with other resources such as Wikidata and allowing more use, curation and redistribution of WikiPathways content.</jats:p>}, number={D1}, journal={Nucleic Acids Research}, publisher={Oxford University Press (OUP)}, author={Martens, Marvin and Ammar, Ammar and Riutta, Anders and Waagmeester, Andra and Slenter, Denise N and Hanspers, Kristina and A. Miller, Ryan and Digles, Daniela and Lopes, Elisson N and Ehrhart, Friederike and Dupuis, Lauren J and Winckers, Laurent A and Coort, Susan L and Willighagen, Egon L and Evelo, Chris T and Pico, Alexander R and Kutmon, Martina}, year={2020}, month=nov, pages={D613–D621}, language={en} "}},
  "10.1093/nar/gkaa1043": {"nature":"Köhler, S. et al. The Human Phenotype Ontology in 2021. Nucleic Acids Research vol. 49 D1207–D1217 (2020).","bibtex":{"type":"misc","record":" title={The Human Phenotype Ontology in 2021}, volume={49}, url={http://dx.doi.org/10.1093/nar/gkaa1043}, DOI={10.1093/nar/gkaa1043}, abstractNote={<jats:title>Abstract</jats:title>\n               <jats:p>The Human Phenotype Ontology (HPO, https://hpo.jax.org) was launched in 2008 to provide a comprehensive logical standard to describe and computationally analyze phenotypic abnormalities found in human disease. The HPO is now a worldwide standard for phenotype exchange. The HPO has grown steadily since its inception due to considerable contributions from clinical experts and researchers from a diverse range of disciplines. Here, we present recent major extensions of the HPO for neurology, nephrology, immunology, pulmonology, newborn screening, and other areas. For example, the seizure subontology now reflects the International League Against Epilepsy (ILAE) guidelines and these enhancements have already shown clinical validity. We present new efforts to harmonize computational definitions of phenotypic abnormalities across the HPO and multiple phenotype ontologies used for animal models of disease. These efforts will benefit software such as Exomiser by improving the accuracy and scope of cross-species phenotype matching. The computational modeling strategy used by the HPO to define disease entities and phenotypic features and distinguish between them is explained in detail.We also report on recent efforts to translate the HPO into indigenous languages. Finally, we summarize recent advances in the use of HPO in electronic health record systems.</jats:p>}, number={D1}, journal={Nucleic Acids Research}, publisher={Oxford University Press (OUP)}, author={Köhler, Sebastian and Gargano, Michael and Matentzoglu, Nicolas and Carmody, Leigh C and Lewis-Smith, David and Vasilevsky, Nicole A and Danis, Daniel and Balagura, Ganna and Baynam, Gareth and Brower, Amy M and Callahan, Tiffany J and Chute, Christopher G and Est, Johanna L and Galer, Peter D and Ganesan, Shiva and Griese, Matthias and Haimel, Matthias and Pazmandi, Julia and Hanauer, Marc and Harris, Nomi L and Hartnett, Michael J and Hastreiter, Maximilian and Hauck, Fabian and He, Yongqun and Jeske, Tim and Kearney, Hugh and Kindle, Gerhard and Klein, Christoph and Knoflach, Katrin and Krause, Roland and Lagorce, David and McMurry, Julie A and Miller, Jillian A and Munoz-Torres, Monica C and Peters, Rebecca L and Rapp, Christina K and Rath, Ana M and Rind, Shahmir A and Rosenberg, Avi Z and Segal, Michael M and Seidel, Markus G and Smedley, Damian and Talmy, Tomer and Thomas, Yarlalu and Wiafe, Samuel A and Xian, Julie and Yüksel, Zafer and Helbig, Ingo and Mungall, Christopher J and Haendel, Melissa A and Robinson, Peter N}, year={2020}, month=dec, pages={D1207–D1217}, language={en} "}},
  "10.1093/nar/gkaa1083": {"nature":"Blake, J. A. et al. Mouse Genome Database (MGD): Knowledgebase for mouse–human comparative biology. Nucleic Acids Research vol. 49 D981–D987 (2020).","bibtex":{"type":"misc","record":" title={Mouse Genome Database (MGD): Knowledgebase for mouse–human comparative biology}, volume={49}, url={http://dx.doi.org/10.1093/nar/gkaa1083}, DOI={10.1093/nar/gkaa1083}, abstractNote={<jats:title>Abstract</jats:title>\n               <jats:p>The Mouse Genome Database (MGD; http://www.informatics.jax.org) is the community model organism knowledgebase for the laboratory mouse, a widely used animal model for comparative studies of the genetic and genomic basis for human health and disease. MGD is the authoritative source for biological reference data related to mouse genes, gene functions, phenotypes and mouse models of human disease. MGD is the primary source for official gene, allele, and mouse strain nomenclature based on the guidelines set by the International Committee on Standardized Nomenclature for Mice. MGD’s biocuration scientists curate information from the biomedical literature and from large and small datasets contributed directly by investigators. In this report we describe significant enhancements to the content and interfaces at MGD, including (i) improvements in the Multi Genome Viewer for exploring the genomes of multiple mouse strains, (ii) inclusion of many more mouse strains and new mouse strain pages with extended query options and (iii) integration of extensive data about mouse strain variants. We also describe improvements to the efficiency of literature curation processes and the implementation of an information portal focused on mouse models and genes for the study of COVID-19.</jats:p>}, number={D1}, journal={Nucleic Acids Research}, publisher={Oxford University Press (OUP)}, author={Blake, Judith A and Baldarelli, Richard and Kadin, James A and Richardson, Joel E and Smith, Cynthia L and Bult, Carol J and Anagnostopoulos, Anna V and Beal, Jon S and Bello, Susan M and Blodgett, Olin and Butler, Nancy E and Campbell, Jeffry and Christie, Karen R and Corbani, Lori E and Dolan, Mary E and Drabkin, Harold J and Flores, Maria and Giannatto, Susan L and Guerra, Angelina and Hale, Paul and Hill, David P and Judd, Jonathan and Law, Meiyee and McAndrews, Monica and Miers, David and Mitchell, Cailey and Motenko, Howie and Ni, Li and Onda, Hiroaki and Ormsby, Janice and Perry, Michelle and Recla, Jill M and Shaw, David and Sitnikov, Dmitry and Tomczuk, Monika and Wilming, Lauren and Zhu, Yunxia ‘Sophia’}, year={2020}, month=nov, pages={D981–D987}, language={en} "}},
  "10.1093/nar/gkab359": {"nature":"Kuleshov, M. V. et al. KEA3: improved kinase enrichment analysis via data integration. Nucleic Acids Research vol. 49 W304–W316 (2021).","bibtex":{"type":"misc","record":" title={KEA3: improved kinase enrichment analysis via data integration}, volume={49}, url={http://dx.doi.org/10.1093/nar/gkab359}, DOI={10.1093/nar/gkab359}, abstractNote={<jats:title>Abstract</jats:title><jats:p>Phosphoproteomics and proteomics experiments capture a global snapshot of the cellular signaling network, but these methods do not directly measure kinase state. Kinase Enrichment Analysis 3 (KEA3) is a webserver application that infers overrepresentation of upstream kinases whose putative substrates are in a user-inputted list of proteins. KEA3 can be applied to analyze data from phosphoproteomics and proteomics studies to predict the upstream kinases responsible for observed differential phosphorylations. The KEA3 background database contains measured and predicted kinase-substrate interactions (KSI), kinase-protein interactions (KPI), and interactions supported by co-expression and co-occurrence data. To benchmark the performance of KEA3, we examined whether KEA3 can predict the perturbed kinase from single-kinase perturbation followed by gene expression experiments, and phosphoproteomics data collected from kinase-targeting small molecules. We show that integrating KSIs and KPIs across data sources to produce a composite ranking improves the recovery of the expected kinase. The KEA3 webserver is available at https://maayanlab.cloud/kea3.</jats:p>}, number={W1}, journal={Nucleic Acids Research}, publisher={Oxford University Press (OUP)}, author={Kuleshov, Maxim V and Xie, Zhuorui and London, Alexandra B K and Yang, Janice and Evangelista, John Erol and Lachmann, Alexander and Shu, Ingrid and Torre, Denis and Ma’ayan, Avi}, year={2021}, month=may, pages={W304–W316}, language={en} "}},
  "10.1093/nar/gkac1000": {"nature":"Szklarczyk, D. et al. The STRING database in 2023: protein–protein association networks and functional enrichment analyses for any sequenced genome of interest. Nucleic Acids Research vol. 51 D638–D646 (2022).","bibtex":{"type":"misc","record":" title={The STRING database in 2023: protein–protein association networks and functional enrichment analyses for any sequenced genome of interest}, volume={51}, url={http://dx.doi.org/10.1093/nar/gkac1000}, DOI={10.1093/nar/gkac1000}, abstractNote={<jats:title>Abstract</jats:title><jats:p>Much of the complexity within cells arises from functional and regulatory interactions among proteins. The core of these interactions is increasingly known, but novel interactions continue to be discovered, and the information remains scattered across different database resources, experimental modalities and levels of mechanistic detail. The STRING database (https://string-db.org/) systematically collects and integrates protein–protein interactions—both physical interactions as well as functional associations. The data originate from a number of sources: automated text mining of the scientific literature, computational interaction predictions from co-expression, conserved genomic context, databases of interaction experiments and known complexes/pathways from curated sources. All of these interactions are critically assessed, scored, and subsequently automatically transferred to less well-studied organisms using hierarchical orthology information. The data can be accessed via the website, but also programmatically and via bulk downloads. The most recent developments in STRING (version 12.0) are: (i) it is now possible to create, browse and analyze a full interaction network for any novel genome of interest, by submitting its complement of encoded proteins, (ii) the co-expression channel now uses variational auto-encoders to predict interactions, and it covers two new sources, single-cell RNA-seq and experimental proteomics data and (iii) the confidence in each experimentally derived interaction is now estimated based on the detection method used, and communicated to the user in the web-interface. Furthermore, STRING continues to enhance its facilities for functional enrichment analysis, which are now fully available also for user-submitted genomes.</jats:p>}, number={D1}, journal={Nucleic Acids Research}, publisher={Oxford University Press (OUP)}, author={Szklarczyk, Damian and Kirsch, Rebecca and Koutrouli, Mikaela and Nastou, Katerina and Mehryary, Farrokh and Hachilif, Radja and Gable, Annika L and Fang, Tao and Doncheva, Nadezhda T and Pyysalo, Sampo and Bork, Peer and Jensen, Lars J and von Mering, Christian}, year={2022}, month=nov, pages={D638–D646}, language={en} "}},
  "10.1093/nar/gkac1010": {"nature":"Sollis, E. et al. The NHGRI-EBI GWAS Catalog: knowledgebase and deposition resource. Nucleic Acids Research vol. 51 D977–D985 (2022).","bibtex":{"type":"misc","record":" title={The NHGRI-EBI GWAS Catalog: knowledgebase and deposition resource}, volume={51}, url={http://dx.doi.org/10.1093/nar/gkac1010}, DOI={10.1093/nar/gkac1010}, abstractNote={<jats:title>Abstract</jats:title>\n               <jats:p>The NHGRI-EBI GWAS Catalog (www.ebi.ac.uk/gwas) is a FAIR knowledgebase providing detailed, structured, standardised and interoperable genome-wide association study (GWAS) data to &amp;gt;200 000 users per year from academic research, healthcare and industry. The Catalog contains variant-trait associations and supporting metadata for &amp;gt;45 000 published GWAS across &amp;gt;5000 human traits, and &amp;gt;40 000 full P-value summary statistics datasets. Content is curated from publications or acquired via author submission of prepublication summary statistics through a new submission portal and validation tool. GWAS data volume has vastly increased in recent years. We have updated our software to meet this scaling challenge and to enable rapid release of submitted summary statistics. The scope of the repository has expanded to include additional data types of high interest to the community, including sequencing-based GWAS, gene-based analyses and copy number variation analyses. Community outreach has increased the number of shared datasets from under-represented traits, e.g. cancer, and we continue to contribute to awareness of the lack of population diversity in GWAS. Interoperability of the Catalog has been enhanced through links to other resources including the Polygenic Score Catalog and the International Mouse Phenotyping Consortium, refinements to GWAS trait annotation, and the development of a standard format for GWAS data.</jats:p>}, number={D1}, journal={Nucleic Acids Research}, publisher={Oxford University Press (OUP)}, author={Sollis, Elliot and Mosaku, Abayomi and Abid, Ala and Buniello, Annalisa and Cerezo, Maria and Gil, Laurent and Groza, Tudor and Güneş, Osman and Hall, Peggy and Hayhurst, James and Ibrahim, Arwa and Ji, Yue and John, Sajo and Lewis, Elizabeth and MacArthur, Jacqueline A L and McMahon, Aoife and Osumi-Sutherland, David and Panoutsopoulou, Kalliope and Pendlington, Zoë and Ramachandran, Santhi and Stefancsik, Ray and Stewart, Jonathan and Whetzel, Patricia and Wilson, Robert and Hindorff, Lucia and Cunningham, Fiona and Lambert, Samuel A and Inouye, Michael and Parkinson, Helen and Harris, Laura W}, year={2022}, month=nov, pages={D977–D985}, language={en} "}},
  "10.1093/nar/gkac328": {"nature":"Evangelista, J. E. et al. SigCom LINCS: data and metadata search engine for a million gene expression signatures. Nucleic Acids Research vol. 50 W697–W709 (2022).","bibtex":{"type":"misc","record":" title={SigCom LINCS: data and metadata search engine for a million gene expression signatures}, volume={50}, url={http://dx.doi.org/10.1093/nar/gkac328}, DOI={10.1093/nar/gkac328}, abstractNote={<jats:title>Abstract</jats:title>\n               <jats:p>Millions of transcriptome samples were generated by the Library of Integrated Network-based Cellular Signatures (LINCS) program. When these data are processed into searchable signatures along with signatures extracted from Genotype-Tissue Expression (GTEx) and Gene Expression Omnibus (GEO), connections between drugs, genes, pathways and diseases can be illuminated. SigCom LINCS is a webserver that serves over a million gene expression signatures processed, analyzed, and visualized from LINCS, GTEx, and GEO. SigCom LINCS is built with Signature Commons, a cloud-agnostic skeleton Data Commons with a focus on serving searchable signatures. SigCom LINCS provides a rapid signature similarity search for mimickers and reversers given sets of up and down genes, a gene set, a single gene, or any search term. Additionally, users of SigCom LINCS can perform a metadata search to find and analyze subsets of signatures and find information about genes and drugs. SigCom LINCS is findable, accessible, interoperable, and reusable (FAIR) with metadata linked to standard ontologies and vocabularies. In addition, all the data and signatures within SigCom LINCS are available via a well-documented API. In summary, SigCom LINCS, available at https://maayanlab.cloud/sigcom-lincs, is a rich webserver resource for accelerating drug and target discovery in systems pharmacology.</jats:p>}, number={W1}, journal={Nucleic Acids Research}, publisher={Oxford University Press (OUP)}, author={Evangelista, John Erol and Clarke, Daniel J B and Xie, Zhuorui and Lachmann, Alexander and Jeon, Minji and Chen, Kerwin and Jagodnik, Kathleen M and Jenkins, Sherry L and Kuleshov, Maxim V and Wojciechowicz, Megan L and Schürer, Stephan C and Medvedovic, Mario and Ma’ayan, Avi}, year={2022}, month=may, pages={W697–W709}, language={en} "}},
  "10.1093/nar/gkac956": {"nature":"Kim, S. et al. PubChem 2023 update. Nucleic Acids Research vol. 51 D1373–D1380 (2022).","bibtex":{"type":"misc","record":" title={PubChem 2023 update}, volume={51}, url={http://dx.doi.org/10.1093/nar/gkac956}, DOI={10.1093/nar/gkac956}, abstractNote={<jats:title>Abstract</jats:title>\n               <jats:p>PubChem (https://pubchem.ncbi.nlm.nih.gov) is a popular chemical information resource that serves a wide range of use cases. In the past two years, a number of changes were made to PubChem. Data from more than 120 data sources was added to PubChem. Some major highlights include: the integration of Google Patents data into PubChem, which greatly expanded the coverage of the PubChem Patent data collection; the creation of the Cell Line and Taxonomy data collections, which provide quick and easy access to chemical information for a given cell line and taxon, respectively; and the update of the bioassay data model. In addition, new functionalities were added to the PubChem programmatic access protocols, PUG-REST and PUG-View, including support for target-centric data download for a given protein, gene, pathway, cell line, and taxon and the addition of the ‘standardize’ option to PUG-REST, which returns the standardized form of an input chemical structure. A significant update was also made to PubChemRDF. The present paper provides an overview of these changes.</jats:p>}, number={D1}, journal={Nucleic Acids Research}, publisher={Oxford University Press (OUP)}, author={Kim, Sunghwan and Chen, Jie and Cheng, Tiejun and Gindulyte, Asta and He, Jia and He, Siqian and Li, Qingliang and Shoemaker, Benjamin A and Thiessen, Paul A and Yu, Bo and Zaslavsky, Leonid and Zhang, Jian and Bolton, Evan E}, year={2022}, month=oct, pages={D1373–D1380}, language={en} "}},
  "10.1093/nar/gkac963": {"nature":"Kanehisa, M., Furumichi, M., Sato, Y., Kawashima, M. & Ishiguro-Watanabe, M. KEGG for taxonomy-based analysis of pathways and genomes. Nucleic Acids Research vol. 51 D587–D592 (2022).","bibtex":{"type":"misc","record":" title={KEGG for taxonomy-based analysis of pathways and genomes}, volume={51}, url={http://dx.doi.org/10.1093/nar/gkac963}, DOI={10.1093/nar/gkac963}, abstractNote={<jats:title>Abstract</jats:title>\n               <jats:p>KEGG (https://www.kegg.jp) is a manually curated database resource integrating various biological objects categorized into systems, genomic, chemical and health information. Each object (database entry) is identified by the KEGG identifier (kid), which generally takes the form of a prefix followed by a five-digit number, and can be retrieved by appending /entry/kid in the URL. The KEGG pathway map viewer, the Brite hierarchy viewer and the newly released KEGG genome browser can be launched by appending /pathway/kid, /brite/kid and /genome/kid, respectively, in the URL. Together with an improved annotation procedure for KO (KEGG Orthology) assignment, an increasing number of eukaryotic genomes have been included in KEGG for better representation of organisms in the taxonomic tree. Multiple taxonomy files are generated for classification of KEGG organisms and viruses, and the Brite hierarchy viewer is used for taxonomy mapping, a variant of Brite mapping in the new KEGG Mapper suite. The taxonomy mapping enables analysis of, for example, how functional links of genes in the pathway and physical links of genes on the chromosome are conserved among organism groups.</jats:p>}, number={D1}, journal={Nucleic Acids Research}, publisher={Oxford University Press (OUP)}, author={Kanehisa, Minoru and Furumichi, Miho and Sato, Yoko and Kawashima, Masayuki and Ishiguro-Watanabe, Mari}, year={2022}, month=oct, pages={D587–D592}, language={en} "}},
  "10.1093/nar/gkac972": {"nature":"Groza, T. et al. The International Mouse Phenotyping Consortium: comprehensive knockout phenotyping underpinning the study of human disease. Nucleic Acids Research vol. 51 D1038–D1045 (2022).","bibtex":{"type":"misc","record":" title={The International Mouse Phenotyping Consortium: comprehensive knockout phenotyping underpinning the study of human disease}, volume={51}, url={http://dx.doi.org/10.1093/nar/gkac972}, DOI={10.1093/nar/gkac972}, abstractNote={<jats:title>Abstract</jats:title>\n               <jats:p>The International Mouse Phenotyping Consortium (IMPC; https://www.mousephenotype.org/) web portal makes available curated, integrated and analysed knockout mouse phenotyping data generated by the IMPC project consisting of 85M data points and over 95,000 statistically significant phenotype hits mapped to human diseases. The IMPC portal delivers a substantial reference dataset that supports the enrichment of various domain-specific projects and databases, as well as the wider research and clinical community, where the IMPC genotype–phenotype knowledge contributes to the molecular diagnosis of patients affected by rare disorders. Data from 9,000 mouse lines and 750 000 images provides vital resources enabling the interpretation of the ignorome, and advancing our knowledge on mammalian gene function and the mechanisms underlying phenotypes associated with human diseases. The resource is widely integrated and the lines have been used in over 4,600 publications indicating the value of the data and the materials.</jats:p>}, number={D1}, journal={Nucleic Acids Research}, publisher={Oxford University Press (OUP)}, author={Groza, Tudor and Gomez, Federico Lopez and Mashhadi, Hamed Haseli and Muñoz-Fuentes, Violeta and Gunes, Osman and Wilson, Robert and Cacheiro, Pilar and Frost, Anthony and Keskivali-Bond, Piia and Vardal, Bora and McCoy, Aaron and Cheng, Tsz Kwan and Santos, Luis and Wells, Sara and Smedley, Damian and Mallon, Ann-Marie and Parkinson, Helen}, year={2022}, month=oct, pages={D1038–D1045}, language={en} "}},
  "10.1093/nar/gkad399": {"nature":"Marino, G. B. et al. GeneRanger and TargetRanger: processed gene and protein expression levels across cells and tissues for target discovery. Nucleic Acids Research vol. 51 W213–W224 (2023).","bibtex":{"type":"misc","record":" title={GeneRanger and TargetRanger: processed gene and protein expression levels across cells and tissues for target discovery}, volume={51}, url={http://dx.doi.org/10.1093/nar/gkad399}, DOI={10.1093/nar/gkad399}, abstractNote={<jats:title>Abstract</jats:title>\n               <jats:p>Several atlasing efforts aim to profile human gene and protein expression across tissues, cell types and cell lines in normal physiology, development and disease. One utility of these resources is to examine the expression of a single gene across all cell types, tissues and cell lines in each atlas. However, there is currently no centralized place that integrates data from several atlases to provide this type of data in a uniform format for visualization, analysis and download, and via an application programming interface. To address this need, GeneRanger is a web server that provides access to processed data about gene and protein expression across normal human cell types, tissues and cell lines from several atlases. At the same time, TargetRanger is a related web server that takes as input RNA-seq data from profiled human cells and tissues, and then compares the uploaded input data to expression levels across the atlases to identify genes that are highly expressed in the input and lowly expressed across normal human cell types and tissues. Identified targets can be filtered by transmembrane or secreted proteins. The results from GeneRanger and TargetRanger are visualized as box and scatter plots, and as interactive tables. GeneRanger and TargetRanger are available from https://generanger.maayanlab.cloud and https://targetranger.maayanlab.cloud, respectively.</jats:p>}, number={W1}, journal={Nucleic Acids Research}, publisher={Oxford University Press (OUP)}, author={Marino, Giacomo B and Ngai, Michael and Clarke, Daniel J B and Fleishman, Reid H and Deng, Eden Z and Xie, Zhuorui and Ahmed, Nasheath and Ma’ayan, Avi}, year={2023}, month=may, pages={W213–W224}, language={en} "}},
  "10.1093/nar/gkad960": {"nature":"Agrawal, A. et al. WikiPathways 2024: next generation pathway database. Nucleic Acids Research vol. 52 D679–D689 (2023).","bibtex":{"type":"misc","record":" title={WikiPathways 2024: next generation pathway database}, volume={52}, url={http://dx.doi.org/10.1093/nar/gkad960}, DOI={10.1093/nar/gkad960}, abstractNote={<jats:title>Abstract</jats:title>\n               <jats:p>WikiPathways (wikipathways.org) is an open-source biological pathway database. Collaboration and open science are pivotal to the success of WikiPathways. Here we highlight the continuing efforts supporting WikiPathways, content growth and collaboration among pathway researchers. As an evolving database, there is a growing need for WikiPathways to address and overcome technical challenges. In this direction, WikiPathways has undergone major restructuring, enabling a renewed approach for sharing and curating pathway knowledge, thus providing stability for the future of community pathway curation. The website has been redesigned to improve and enhance user experience. This next generation of WikiPathways continues to support existing features while improving maintainability of the database and facilitating community input by providing new functionality and leveraging automation.</jats:p>}, number={D1}, journal={Nucleic Acids Research}, publisher={Oxford University Press (OUP)}, author={Agrawal, Ayushi and Balcı, Hasan and Hanspers, Kristina and Coort, Susan L and Martens, Marvin and Slenter, Denise N and Ehrhart, Friederike and Digles, Daniela and Waagmeester, Andra and Wassink, Isabel and Abbassi-Daloii, Tooba and Lopes, Elisson N and Iyer, Aishwarya and Acosta, Javier Millán and Willighagen, Lars G and Nishida, Kozo and Riutta, Anders and Basaric, Helena and Evelo, Chris T and Willighagen, Egon L and Kutmon, Martina and Pico, Alexander R}, year={2023}, month=nov, pages={D679–D689}, language={en} "}},
  "10.1093/nar/gks1114": {"nature":"Wu, C., MacLeod, I. & Su, A. I. BioGPS and MyGene.info: organizing online, gene-centric information. Nucleic Acids Research vol. 41 D561–D565 (2012).","bibtex":{"type":"misc","record":" title={BioGPS and MyGene.info: organizing online, gene-centric information}, volume={41}, url={http://dx.doi.org/10.1093/nar/gks1114}, DOI={10.1093/nar/gks1114}, number={D1}, journal={Nucleic Acids Research}, publisher={Oxford University Press (OUP)}, author={Wu, Chunlei and MacLeod, Ian and Su, Andrew I.}, year={2012}, month=nov, pages={D561–D565}, language={en} "}},
  "10.1093/nar/gks1193": {"nature":"Barrett, T. et al. NCBI GEO: archive for functional genomics data sets—update. Nucleic Acids Research vol. 41 D991–D995 (2012).","bibtex":{"type":"misc","record":" title={NCBI GEO: archive for functional genomics data sets—update}, volume={41}, url={http://dx.doi.org/10.1093/nar/gks1193}, DOI={10.1093/nar/gks1193}, number={D1}, journal={Nucleic Acids Research}, publisher={Oxford University Press (OUP)}, author={Barrett, Tanya and Wilhite, Stephen E. and Ledoux, Pierre and Evangelista, Carlos and Kim, Irene F. and Tomashevsky, Maxim and Marshall, Kimberly A. and Phillippy, Katherine H. and Sherman, Patti M. and Holko, Michelle and Yefanov, Andrey and Lee, Hyeseung and Zhang, Naigong and Robertson, Cynthia L. and Serova, Nadezhda and Davis, Sean and Soboleva, Alexandra}, year={2012}, month=nov, pages={D991–D995}, language={en} "}},
  "10.1093/nar/gku1003": {"nature":"Szklarczyk, D. et al. STRING v10: protein–protein interaction networks, integrated over the tree of life. Nucleic Acids Research vol. 43 D447–D452 (2014).","bibtex":{"type":"misc","record":" title={STRING v10: protein–protein interaction networks, integrated over the tree of life}, volume={43}, url={http://dx.doi.org/10.1093/nar/gku1003}, DOI={10.1093/nar/gku1003}, number={D1}, journal={Nucleic Acids Research}, publisher={Oxford University Press (OUP)}, author={Szklarczyk, Damian and Franceschini, Andrea and Wyder, Stefan and Forslund, Kristoffer and Heller, Davide and Huerta-Cepas, Jaime and Simonovic, Milan and Roth, Alexander and Santos, Alberto and Tsafou, Kalliopi P. and Kuhn, Michael and Bork, Peer and Jensen, Lars J. and von Mering, Christian}, year={2014}, month=oct, pages={D447–D452}, language={en} "}},
  "10.1093/nar/gkv007": {"nature":"Ritchie, M. E. et al. limma powers differential expression analyses for RNA-sequencing and microarray studies. Nucleic Acids Research vol. 43 e47–e47 (2015).","bibtex":{"type":"misc","record":" title={limma powers differential expression analyses for RNA-sequencing and microarray studies}, volume={43}, url={http://dx.doi.org/10.1093/nar/gkv007}, DOI={10.1093/nar/gkv007}, number={7}, journal={Nucleic Acids Research}, publisher={Oxford University Press (OUP)}, author={Ritchie, Matthew E. and Phipson, Belinda and Wu, Di and Hu, Yifang and Law, Charity W. and Shi, Wei and Smyth, Gordon K.}, year={2015}, month=jan, pages={e47–e47}, language={en} "}},
  "10.1093/nar/gkx1153": {"nature":"Landrum, M. J. et al. ClinVar: improving access to variant interpretations and supporting evidence. Nucleic Acids Research vol. 46 D1062–D1067 (2017).","bibtex":{"type":"misc","record":" title={ClinVar: improving access to variant interpretations and supporting evidence}, volume={46}, url={http://dx.doi.org/10.1093/nar/gkx1153}, DOI={10.1093/nar/gkx1153}, number={D1}, journal={Nucleic Acids Research}, publisher={Oxford University Press (OUP)}, author={Landrum, Melissa J and Lee, Jennifer M and Benson, Mark and Brown, Garth R and Chao, Chen and Chitipiralla, Shanmuga and Gu, Baoshan and Hart, Jennifer and Hoffman, Douglas and Jang, Wonhee and Karapetyan, Karen and Katz, Kenneth and Liu, Chunlei and Maddipatla, Zenith and Malheiro, Adriana and McDaniel, Kurt and Ovetsky, Michael and Riley, George and Zhou, George and Holmes, J Bradley and Kattman, Brandi L and Maglott, Donna R}, year={2017}, month=nov, pages={D1062–D1067}, language={en} "}},
  "10.1093/nar/gkz393": {"nature":"Lachmann, A. et al. Geneshot: search engine for ranking genes from arbitrary text queries. Nucleic Acids Research vol. 47 W571–W577 (2019).","bibtex":{"type":"misc","record":" title={Geneshot: search engine for ranking genes from arbitrary text queries}, volume={47}, url={http://dx.doi.org/10.1093/nar/gkz393}, DOI={10.1093/nar/gkz393}, abstractNote={<jats:title>Abstract</jats:title>\n               <jats:p>The frequency by which genes are studied correlates with the prior knowledge accumulated about them. This leads to an imbalance in research attention where some genes are highly investigated while others are ignored. Geneshot is a search engine developed to illuminate this gap and to promote attention to the under-studied genome. Through a simple web interface, Geneshot enables researchers to enter arbitrary search terms, to receive ranked lists of genes relevant to the search terms. Returned ranked gene lists contain genes that were previously published in association with the search terms, as well as genes predicted to be associated with the terms based on data integration from multiple sources. The search results are presented with interactive visualizations. To predict gene function, Geneshot utilizes gene–gene similarity matrices from processed RNA-seq data, or from gene–gene co-occurrence data obtained from multiple sources. In addition, Geneshot can be used to analyze the novelty of gene sets and augment gene sets with additional relevant genes. The Geneshot web-server and API are freely and openly available from https://amp.pharm.mssm.edu/geneshot.</jats:p>}, number={W1}, journal={Nucleic Acids Research}, publisher={Oxford University Press (OUP)}, author={Lachmann, Alexander and Schilder, Brian M and Wojciechowicz, Megan L and Torre, Denis and Kuleshov, Maxim V and Keenan, Alexandra B and Ma’ayan, Avi}, year={2019}, month=may, pages={W571–W577}, language={en} "}},
  "10.1093/nar/gkz446": {"nature":"Keenan, A. B. et al. ChEA3: transcription factor enrichment analysis by orthogonal omics integration. Nucleic Acids Research vol. 47 W212–W224 (2019).","bibtex":{"type":"misc","record":" title={ChEA3: transcription factor enrichment analysis by orthogonal omics integration}, volume={47}, url={http://dx.doi.org/10.1093/nar/gkz446}, DOI={10.1093/nar/gkz446}, abstractNote={<jats:title>Abstract</jats:title><jats:p>Identifying the transcription factors (TFs) responsible for observed changes in gene expression is an important step in understanding gene regulatory networks. ChIP-X Enrichment Analysis 3 (ChEA3) is a transcription factor enrichment analysis tool that ranks TFs associated with user-submitted gene sets. The ChEA3 background database contains a collection of gene set libraries generated from multiple sources including TF–gene co-expression from RNA-seq studies, TF–target associations from ChIP-seq experiments, and TF–gene co-occurrence computed from crowd-submitted gene lists. Enrichment results from these distinct sources are integrated to generate a composite rank that improves the prediction of the correct upstream TF compared to ranks produced by individual libraries. We compare ChEA3 with existing TF prediction tools and show that ChEA3 performs better. By integrating the ChEA3 libraries, we illuminate general transcription factor properties such as whether the TF behaves as an activator or a repressor. The ChEA3 web-server is available from https://amp.pharm.mssm.edu/ChEA3.</jats:p>}, number={W1}, journal={Nucleic Acids Research}, publisher={Oxford University Press (OUP)}, author={Keenan, Alexandra B and Torre, Denis and Lachmann, Alexander and Leong, Ariel K and Wojciechowicz, Megan L and Utti, Vivian and Jagodnik, Kathleen M and Kropiwnicki, Eryk and Wang, Zichen and Ma’ayan, Avi}, year={2019}, month=may, pages={W212–W224}, language={en} "}},
  "10.1101/2021.12.16.473007": {"nature":"Virshup, I., Rybakov, S., Theis, F. J., Angerer, P. & Wolf, F. A. anndata: Annotated data. (2021) doi:10.1101/2021.12.16.473007.","bibtex":{"type":"misc","record":" title={anndata: Annotated data}, url={http://dx.doi.org/10.1101/2021.12.16.473007}, DOI={10.1101/2021.12.16.473007}, abstractNote={<jats:title>Summary</jats:title><jats:p>anndata is a Python package for handling annotated data matrices in memory and on disk (<jats:ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"http://github.com/theislab/anndata\">github.com/theislab/anndata</jats:ext-link>), positioned between pandas and xarray. anndata offers a broad range of computationally efficient features including, among others, sparse data support, lazy operations, and a PyTorch interface.</jats:p><jats:sec><jats:title>Statement of need</jats:title><jats:p>Generating insight from high-dimensional data matrices typically works through training models that annotate observations and variables via low-dimensional representations. In exploratory data analysis, this involves<jats:italic>iterative</jats:italic>training and analysis using original and learned annotations and task-associated representations. anndata offers a canonical data structure for book-keeping these, which is neither addressed by pandas (McKinney, 2010), nor xarray (Hoyer &amp; Hamman, 2017), nor commonly-used modeling packages like scikit-learn (Pedregosa et al., 2011).</jats:p></jats:sec>}, publisher={Cold Spring Harbor Laboratory}, author={Virshup, Isaac and Rybakov, Sergei and Theis, Fabian J. and Angerer, Philipp and Wolf, F. Alexander}, year={2021}, month=dec "}},
  "10.1101/382937": {"nature":"Lachmann, A., Xie, Z. & Ma’ayan, A. Elysium: RNA-seq Alignment in the Cloud. (2018) doi:10.1101/382937.","bibtex":{"type":"misc","record":" title={Elysium: RNA-seq Alignment in the Cloud}, url={http://dx.doi.org/10.1101/382937}, DOI={10.1101/382937}, abstractNote={<jats:sec><jats:title>Motivation</jats:title><jats:p>RNA-sequencing (RNA-seq) is currently the leading technology for genome-wide transcript quantification. Mapping the raw reads to transcript and gene level counts can be achieved by a variety of aligners and pipelines. The diversity of processing options reduces interoperability. In addition, the alignment step requires significant computational resources and basic programming knowledge. Elysium enables users of all skill levels to perform a uniform and free RNA-seq alignment in the cloud.</jats:p></jats:sec><jats:sec><jats:title>Results</jats:title><jats:p>The Elysium infrastructure is comprised of four components: A file upload API that enables storage of FASTQ files on Amazon S3 without Amazon credentials; an API to handle the cloud alignment job scheduling for uploaded files; and a graphical user interface (GUI) to provide intuitive access to users that do not have command-line access skills.</jats:p></jats:sec><jats:sec><jats:title>Availability</jats:title><jats:p>The Elysium source code is available under the Apache Licence 2.0 on GitHub at: <jats:ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://github.com/maayanlab/elysium\">https://github.com/maayanlab/elysium</jats:ext-link></jats:p><jats:p>The service of cloud based RNA-seq alignment is freely accessible through the Elysium GUI at: <jats:ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"http://elysium.cloud\">http://elysium.cloud</jats:ext-link></jats:p></jats:sec>}, publisher={Cold Spring Harbor Laboratory}, author={Lachmann, Alexander and Xie, Zhuorui and Ma’ayan, Avi}, year={2018}, month=aug "}},
  "10.1109/TVCG.2014.2346248": {"nature":"Lex, A., Gehlenborg, N., Strobelt, H., Vuillemot, R. & Pfister, H. UpSet: Visualization of Intersecting Sets. IEEE Transactions on Visualization and Computer Graphics vol. 20 1983–1992 (2014).","bibtex":{"type":"misc","record":" title={UpSet: Visualization of Intersecting Sets}, volume={20}, url={http://dx.doi.org/10.1109/TVCG.2014.2346248}, DOI={10.1109/tvcg.2014.2346248}, number={12}, journal={IEEE Transactions on Visualization and Computer Graphics}, publisher={Institute of Electrical and Electronics Engineers (IEEE)}, author={Lex, Alexander and Gehlenborg, Nils and Strobelt, Hendrik and Vuillemot, Romain and Pfister, Hanspeter}, year={2014}, month=dec, pages={1983–1992} "}},
  "10.1186/1471-2105-15-79": {"nature":"Clark, N. R. et al. The characteristic direction: a geometrical approach to identify differentially expressed genes. BMC Bioinformatics vol. 15 (2014).","bibtex":{"type":"misc","record":" title={The characteristic direction: a geometrical approach to identify differentially expressed genes}, volume={15}, url={http://dx.doi.org/10.1186/1471-2105-15-79}, DOI={10.1186/1471-2105-15-79}, number={1}, journal={BMC Bioinformatics}, publisher={Springer Science and Business Media LLC}, author={Clark, Neil R and Hu, Kevin S and Feldmann, Axel S and Kou, Yan and Chen, Edward Y and Duan, Qiaonan and Ma’ayan, Avi}, year={2014}, month=mar, language={en} "}},
  "10.1186/gb-2013-14-9-r95": {"nature":"Rapaport, F. et al. Comprehensive evaluation of differential gene expression analysis methods for RNA-seq data. Genome Biology vol. 14 R95 (2013).","bibtex":{"type":"misc","record":" title={Comprehensive evaluation of differential gene expression analysis methods for RNA-seq data}, volume={14}, url={http://dx.doi.org/10.1186/gb-2013-14-9-r95}, DOI={10.1186/gb-2013-14-9-r95}, number={9}, journal={Genome Biology}, publisher={Springer Science and Business Media LLC}, author={Rapaport, Franck and Khanin, Raya and Liang, Yupu and Pirun, Mono and Krek, Azra and Zumbo, Paul and Mason, Christopher E and Socci, Nicholas D and Betel, Doron}, year={2013}, pages={R95}, language={en} "}},
  "10.1186/gb-2014-15-2-r29": {"nature":"Law, C. W., Chen, Y., Shi, W. & Smyth, G. K. voom: precision weights unlock linear model analysis tools for RNA-seq read counts. Genome Biology vol. 15 R29 (2014).","bibtex":{"type":"misc","record":" title={voom: precision weights unlock linear model analysis tools for RNA-seq read counts}, volume={15}, url={http://dx.doi.org/10.1186/gb-2014-15-2-r29}, DOI={10.1186/gb-2014-15-2-r29}, number={2}, journal={Genome Biology}, publisher={Springer Science and Business Media LLC}, author={Law, Charity W and Chen, Yunshun and Shi, Wei and Smyth, Gordon K}, year={2014}, pages={R29}, language={en} "}},
  "10.1186/s12859-022-04590-5": {"nature":"Kropiwnicki, E. et al. DrugShot: querying biomedical search terms to retrieve prioritized lists of small molecules. BMC Bioinformatics vol. 23 (2022).","bibtex":{"type":"misc","record":" title={DrugShot: querying biomedical search terms to retrieve prioritized lists of small molecules}, volume={23}, url={http://dx.doi.org/10.1186/s12859-022-04590-5}, DOI={10.1186/s12859-022-04590-5}, abstractNote={<jats:title>Abstract</jats:title><jats:sec><jats:title>Background</jats:title><jats:p>PubMed contains millions of abstracts that co-mention terms that describe drugs with other biomedical terms such as genes or diseases. Unique opportunities exist for leveraging these co-mentions by integrating them with other drug-drug similarity resources such as the Library of Integrated Network-based Cellular Signatures (LINCS) L1000 signatures to develop novel hypotheses.</jats:p></jats:sec><jats:sec><jats:title>Results</jats:title><jats:p>DrugShot is a web-based server application and an Appyter that enables users to enter any biomedical search term into a simple input form to receive ranked lists of drugs and other small molecules based on their relevance to the search term. To produce ranked lists of small molecules, DrugShot cross-references returned PubMed identifiers (PMIDs) with DrugRIF or AutoRIF, which are curated resources of drug-PMID associations, to produce an associated small molecule list where each small molecule is ranked according to total co-mentions with the search term from shared PubMed IDs. Additionally, using two types of drug-drug similarity matrices, lists of small molecules are predicted to be associated with the search term. Such predictions are based on literature co-mentions and signature similarity from LINCS L1000 drug-induced gene expression profiles.</jats:p></jats:sec><jats:sec><jats:title>Conclusions</jats:title><jats:p>DrugShot prioritizes drugs and small molecules associated with biomedical search terms. In addition to listing known associations, DrugShot predicts additional drugs and small molecules related to any search term. Hence, DrugShot can be used to prioritize drugs and preclinical compounds for drug repurposing and suggest indications and adverse events for preclinical compounds. DrugShot is freely and openly available at:<jats:ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://maayanlab.cloud/drugshot\">https://maayanlab.cloud/drugshot</jats:ext-link>and<jats:ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://appyters.maayanlab.cloud/#/DrugShot\">https://appyters.maayanlab.cloud/#/DrugShot</jats:ext-link>.</jats:p></jats:sec>}, number={1}, journal={BMC Bioinformatics}, publisher={Springer Science and Business Media LLC}, author={Kropiwnicki, Eryk and Lachmann, Alexander and Clarke, Daniel J. B. and Xie, Zhuorui and Jagodnik, Kathleen M. and Ma’ayan, Avi}, year={2022}, month=feb, language={en} "}},
  "10.1186/s12859-022-04895-5": {"nature":"Jeon, M. et al. Transforming L1000 profiles to RNA-seq-like profiles with deep learning. BMC Bioinformatics vol. 23 (2022).","bibtex":{"type":"misc","record":" title={Transforming L1000 profiles to RNA-seq-like profiles with deep learning}, volume={23}, url={http://dx.doi.org/10.1186/s12859-022-04895-5}, DOI={10.1186/s12859-022-04895-5}, abstractNote={<jats:title>Abstract</jats:title><jats:p>The L1000 technology, a cost-effective high-throughput transcriptomics technology, has been applied to profile a collection of human cell lines for their gene expression response to &gt; 30,000 chemical and genetic perturbations. In total, there are currently over 3 million available L1000 profiles. Such a dataset is invaluable for the discovery of drug and target candidates and for inferring mechanisms of action for small molecules. The L1000 assay only measures the mRNA expression of 978 landmark genes while 11,350 additional genes are computationally reliably inferred. The lack of full genome coverage limits knowledge discovery for half of the human protein coding genes, and the potential for integration with other transcriptomics profiling data. Here we present a Deep Learning two-step model that transforms L1000 profiles to RNA-seq-like profiles. The input to the model are the measured 978 landmark genes while the output is a vector of 23,614 RNA-seq-like gene expression profiles. The model first transforms the landmark genes into RNA-seq-like 978 gene profiles using a modified CycleGAN model applied to unpaired data. The transformed 978 RNA-seq-like landmark genes are then extrapolated into the full genome space with a fully connected neural network model. The two-step model achieves 0.914 Pearson’s correlation coefficients and 1.167 root mean square errors when tested on a published paired L1000/RNA-seq dataset produced by the LINCS and GTEx programs. The processed RNA-seq-like profiles are made available for download, signature search, and gene centric reverse search with unique case studies.</jats:p>}, number={1}, journal={BMC Bioinformatics}, publisher={Springer Science and Business Media LLC}, author={Jeon, Minji and Xie, Zhuorui and Evangelista, John E. and Wojciechowicz, Megan L. and Clarke, Daniel J. B. and Ma’ayan, Avi}, year={2022}, month=sep, language={en} "}},
  "10.1186/s13059-016-0953-9": {"nature":"Xin, J. et al. High-performance web services for querying gene and variant annotation. Genome Biology vol. 17 (2016).","bibtex":{"type":"misc","record":" title={High-performance web services for querying gene and variant annotation}, volume={17}, url={http://dx.doi.org/10.1186/s13059-016-0953-9}, DOI={10.1186/s13059-016-0953-9}, number={1}, journal={Genome Biology}, publisher={Springer Science and Business Media LLC}, author={Xin, Jiwen and Mark, Adam and Afrasiabi, Cyrus and Tsueng, Ginger and Juchler, Moritz and Gopal, Nikhil and Stupp, Gregory S. and Putman, Timothy E. and Ainscough, Benjamin J. and Griffith, Obi L. and Torkamani, Ali and Whetzel, Patricia L. and Mungall, Christopher J. and Mooney, Sean D. and Su, Andrew I. and Wu, Chunlei}, year={2016}, month=may, language={en} "}},
  "10.1371/journal.pcbi.1009551": {"nature":"Thistlethwaite, L. R. et al. Correction: CTD: An information-theoretic algorithm to interpret sets of metabolomic and transcriptomic perturbations in the context of graphical models. PLOS Computational Biology vol. 17 e1009551 (2021).","bibtex":{"type":"misc","record":" title={Correction: CTD: An information-theoretic algorithm to interpret sets of metabolomic and transcriptomic perturbations in the context of graphical models}, volume={17}, url={http://dx.doi.org/10.1371/journal.pcbi.1009551}, DOI={10.1371/journal.pcbi.1009551}, number={10}, journal={PLOS Computational Biology}, publisher={Public Library of Science (PLoS)}, author={Thistlethwaite, Lillian R. and Petrosyan, Varduhi and Li, Xiqi and Miller, Marcus J. and Elsea, Sarah H. and Milosavljevic, Aleksandar}, year={2021}, month=oct, pages={e1009551}, language={en} "}},
  "10.1371/journal.pmed.1001779": {"nature":"Sudlow, C. et al. UK Biobank: An Open Access Resource for Identifying the Causes of a Wide Range of Complex Diseases of Middle and Old Age. PLOS Medicine vol. 12 e1001779 (2015).","bibtex":{"type":"misc","record":" title={UK Biobank: An Open Access Resource for Identifying the Causes of a Wide Range of Complex Diseases of Middle and Old Age}, volume={12}, url={http://dx.doi.org/10.1371/journal.pmed.1001779}, DOI={10.1371/journal.pmed.1001779}, number={3}, journal={PLOS Medicine}, publisher={Public Library of Science (PLoS)}, author={Sudlow, Cathie and Gallacher, John and Allen, Naomi and Beral, Valerie and Burton, Paul and Danesh, John and Downey, Paul and Elliott, Paul and Green, Jane and Landray, Martin and Liu, Bette and Matthews, Paul and Ong, Giok and Pell, Jill and Silman, Alan and Young, Alan and Sprosen, Tim and Peakman, Tim and Collins, Rory}, year={2015}, month=mar, pages={e1001779}, language={en} "}},
  "10.48550/arXiv.1802.03426": {"nature":"McInnes, L., Healy, J. & Melville, J. UMAP: Uniform Manifold Approximation and Projection for Dimension Reduction. Preprint at https://doi.org/10.48550/ARXIV.1802.03426 (2018).","bibtex":{"type":"article","record":" title={UMAP: Uniform Manifold Approximation and Projection for Dimension Reduction}, url={https://arxiv.org/abs/1802.03426}, DOI={10.48550/ARXIV.1802.03426}, abstractNote={UMAP (Uniform Manifold Approximation and Projection) is a novel manifold learning technique for dimension reduction. UMAP is constructed from a theoretical framework based in Riemannian geometry and algebraic topology. The result is a practical scalable algorithm that applies to real world data. The UMAP algorithm is competitive with t-SNE for visualization quality, and arguably preserves more of the global structure with superior run time performance. Furthermore, UMAP has no computational restrictions on embedding dimension, making it viable as a general purpose dimension reduction technique for machine learning.}, publisher={arXiv}, author={McInnes, Leland and Healy, John and Melville, James}, year={2018} "}},
  "10.5281/zenodo.4016732": {"nature":", Fedor. Gecko984/Supervenn: Add Some Tests for Supervenn(). (Zenodo, 2020). doi:10.5281/ZENODO.4016732.","bibtex":{"type":"book","record":" title={gecko984/supervenn: add some tests for supervenn()}, url={https://zenodo.org/record/4016732}, DOI={10.5281/ZENODO.4016732}, abstractNote={supervenn: precise and easy-to-read multiple sets visualization in Python}, publisher={Zenodo}, author={, Fedor}, year={2020}, month=sep "}},
}