{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "source": [
    "import csv\n",
    "import json\n",
    "import pandas as pd\n",
    "import networkx as nx\n",
    "from collections import Counter"
   ],
   "outputs": [],
   "metadata": {}
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "source": [
    "g = nx.MultiDiGraph()"
   ],
   "outputs": [],
   "metadata": {}
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "source": [
    "df_metanodes = pd.read_csv('data/metanodes.tsv', sep='\\t')"
   ],
   "outputs": [],
   "metadata": {}
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "source": [
    "df_metanodes"
   ],
   "outputs": [
    {
     "output_type": "execute_result",
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>id</th>\n",
       "      <th>type</th>\n",
       "      <th>input</th>\n",
       "      <th>output</th>\n",
       "      <th>label</th>\n",
       "      <th>description</th>\n",
       "      <th>color</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>ARCHS4SignatureResults</td>\n",
       "      <td>Data</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ARCHS4 Signature Search Results</td>\n",
       "      <td>ARCHS4 Signatures Query Results</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>AlleleRegistryExternalRecordsTable</td>\n",
       "      <td>Data</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>AlleleRegistryExternalRecordsTable</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>AlleleSpecificEvidencesTable</td>\n",
       "      <td>Data</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Allele Specific Evidences Table</td>\n",
       "      <td>A table of allele specific evidences</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>AnnData</td>\n",
       "      <td>Data</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Annotated data</td>\n",
       "      <td>A gene count matrix paired with sample annotat...</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>BokehPlot</td>\n",
       "      <td>Data</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Bokeh Plot</td>\n",
       "      <td>A figure created with the [Bokeh Library](http...</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>351</th>\n",
       "      <td>VariantInfoFromVariantTerm</td>\n",
       "      <td>Resolver</td>\n",
       "      <td>{\"variant\":\"Term[Variant]\"}</td>\n",
       "      <td>VariantInfo</td>\n",
       "      <td>Resolve Variant Info from Term</td>\n",
       "      <td>Resolve variant info (Allele registry API) fro...</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>352</th>\n",
       "      <td>VisualizeLibrarySizes</td>\n",
       "      <td>Resolver</td>\n",
       "      <td>{\"matrix\":\"GeneCountMatrix\"}</td>\n",
       "      <td>PlotlyPlot</td>\n",
       "      <td>Library Size Bar Plot from Gene Count Matrix</td>\n",
       "      <td>Construct a bar plot which displays the total ...</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>353</th>\n",
       "      <td>VisualizeLibrarySizesFromAnnData</td>\n",
       "      <td>Resolver</td>\n",
       "      <td>{\"anndata\":\"AnnData\"}</td>\n",
       "      <td>PlotlyPlot</td>\n",
       "      <td>Library Size Bar Plot from AnnData File</td>\n",
       "      <td>Construct a bar plot which displays the total ...</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>354</th>\n",
       "      <td>VolcanoPlot</td>\n",
       "      <td>Resolver</td>\n",
       "      <td>{\"sig\":\"GeneSignature\"}</td>\n",
       "      <td>PlotlyPlot</td>\n",
       "      <td>Volcano Plot from Differential Expression Table</td>\n",
       "      <td>Construct a scatter plot which displays the lo...</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>355</th>\n",
       "      <td>ZScoreNormalizeGeneCountMatrix</td>\n",
       "      <td>Resolver</td>\n",
       "      <td>{\"matrix\":\"GeneCountMatrix\"}</td>\n",
       "      <td>GeneCountMatrix</td>\n",
       "      <td>Z Score Normalize A Gene Count Matrix</td>\n",
       "      <td>Z-score normalize a gene count matrix, return ...</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>356 rows × 7 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                     id      type  \\\n",
       "0                ARCHS4SignatureResults      Data   \n",
       "1    AlleleRegistryExternalRecordsTable      Data   \n",
       "2          AlleleSpecificEvidencesTable      Data   \n",
       "3                               AnnData      Data   \n",
       "4                             BokehPlot      Data   \n",
       "..                                  ...       ...   \n",
       "351          VariantInfoFromVariantTerm  Resolver   \n",
       "352               VisualizeLibrarySizes  Resolver   \n",
       "353    VisualizeLibrarySizesFromAnnData  Resolver   \n",
       "354                         VolcanoPlot  Resolver   \n",
       "355      ZScoreNormalizeGeneCountMatrix  Resolver   \n",
       "\n",
       "                            input           output  \\\n",
       "0                             NaN              NaN   \n",
       "1                             NaN              NaN   \n",
       "2                             NaN              NaN   \n",
       "3                             NaN              NaN   \n",
       "4                             NaN              NaN   \n",
       "..                            ...              ...   \n",
       "351   {\"variant\":\"Term[Variant]\"}      VariantInfo   \n",
       "352  {\"matrix\":\"GeneCountMatrix\"}       PlotlyPlot   \n",
       "353         {\"anndata\":\"AnnData\"}       PlotlyPlot   \n",
       "354       {\"sig\":\"GeneSignature\"}       PlotlyPlot   \n",
       "355  {\"matrix\":\"GeneCountMatrix\"}  GeneCountMatrix   \n",
       "\n",
       "                                               label  \\\n",
       "0                    ARCHS4 Signature Search Results   \n",
       "1                 AlleleRegistryExternalRecordsTable   \n",
       "2                    Allele Specific Evidences Table   \n",
       "3                                     Annotated data   \n",
       "4                                         Bokeh Plot   \n",
       "..                                               ...   \n",
       "351                   Resolve Variant Info from Term   \n",
       "352     Library Size Bar Plot from Gene Count Matrix   \n",
       "353          Library Size Bar Plot from AnnData File   \n",
       "354  Volcano Plot from Differential Expression Table   \n",
       "355            Z Score Normalize A Gene Count Matrix   \n",
       "\n",
       "                                           description color  \n",
       "0                      ARCHS4 Signatures Query Results   NaN  \n",
       "1                                                  NaN   NaN  \n",
       "2                 A table of allele specific evidences   NaN  \n",
       "3    A gene count matrix paired with sample annotat...   NaN  \n",
       "4    A figure created with the [Bokeh Library](http...   NaN  \n",
       "..                                                 ...   ...  \n",
       "351  Resolve variant info (Allele registry API) fro...   NaN  \n",
       "352  Construct a bar plot which displays the total ...   NaN  \n",
       "353  Construct a bar plot which displays the total ...   NaN  \n",
       "354  Construct a scatter plot which displays the lo...   NaN  \n",
       "355  Z-score normalize a gene count matrix, return ...   NaN  \n",
       "\n",
       "[356 rows x 7 columns]"
      ]
     },
     "metadata": {},
     "execution_count": 4
    }
   ],
   "metadata": {}
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "source": [
    "for id, node in df_metanodes.iterrows():\n",
    "  g.add_node(id, **node.to_dict())\n",
    "\n",
    "metanodes = df_metanodes.set_index('id')\n",
    "metanode_lookup = df_metanodes.reset_index().set_index('id')['index'].to_dict()"
   ],
   "outputs": [],
   "metadata": {}
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "source": [
    "df = pd.read_csv('data/dump.csv', header=None, names=['id', 'parent', 'process_type', 'input_data_type'])"
   ],
   "outputs": [],
   "metadata": {}
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "source": [
    "df"
   ],
   "outputs": [
    {
     "output_type": "execute_result",
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>id</th>\n",
       "      <th>parent</th>\n",
       "      <th>process_type</th>\n",
       "      <th>input_data_type</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>3af79be2-c018-4926-6298-076b7c8b4dbb</td>\n",
       "      <td>6643d55a-355e-3371-c6b7-45fc96a74513</td>\n",
       "      <td>OneScoredT[Scored[Gene]]</td>\n",
       "      <td>Term[Gene]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>15293aeb-7703-3a75-e4d4-bf8028159a0c</td>\n",
       "      <td>NaN</td>\n",
       "      <td>FileInput</td>\n",
       "      <td>FileURL</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>ecef3f2a-4357-88b6-97d5-4199f370d6d5</td>\n",
       "      <td>15293aeb-7703-3a75-e4d4-bf8028159a0c</td>\n",
       "      <td>GeneCountMatrixFromFile</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>031adf2a-6035-1bd4-1d7e-22db34991c80</td>\n",
       "      <td>NaN</td>\n",
       "      <td>FileInput</td>\n",
       "      <td>FileURL</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>20946800-c926-b0cb-2ea4-2416250ca2c4</td>\n",
       "      <td>031adf2a-6035-1bd4-1d7e-22db34991c80</td>\n",
       "      <td>GeneCountMatrixFromFile</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2435</th>\n",
       "      <td>c689c679-f02a-3f90-06b1-873a5d94cb53</td>\n",
       "      <td>e4970b43-04fd-0362-d73a-bab64815ae3b</td>\n",
       "      <td>ExtractEnrichrTermSearch[LINCS_L1000_Chem_Pert...</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2436</th>\n",
       "      <td>5a11a546-052a-0a0c-799c-ec8fcfaef122</td>\n",
       "      <td>c689c679-f02a-3f90-06b1-873a5d94cb53</td>\n",
       "      <td>EnrichrSetTToGMT[Drug]</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2437</th>\n",
       "      <td>725f2705-8a6f-1d6e-53b3-142ae21e108c</td>\n",
       "      <td>5a11a546-052a-0a0c-799c-ec8fcfaef122</td>\n",
       "      <td>GenesetsToGMT</td>\n",
       "      <td>GMT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2438</th>\n",
       "      <td>c06b0d4b-ef84-4a56-0f60-afecab26181c</td>\n",
       "      <td>725f2705-8a6f-1d6e-53b3-142ae21e108c</td>\n",
       "      <td>GMTConcatenate</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2439</th>\n",
       "      <td>2255f6bb-0003-4245-3a27-ed67aa101303</td>\n",
       "      <td>c06b0d4b-ef84-4a56-0f60-afecab26181c</td>\n",
       "      <td>SupervennFromGMT</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2440 rows × 4 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                        id  \\\n",
       "0     3af79be2-c018-4926-6298-076b7c8b4dbb   \n",
       "1     15293aeb-7703-3a75-e4d4-bf8028159a0c   \n",
       "2     ecef3f2a-4357-88b6-97d5-4199f370d6d5   \n",
       "3     031adf2a-6035-1bd4-1d7e-22db34991c80   \n",
       "4     20946800-c926-b0cb-2ea4-2416250ca2c4   \n",
       "...                                    ...   \n",
       "2435  c689c679-f02a-3f90-06b1-873a5d94cb53   \n",
       "2436  5a11a546-052a-0a0c-799c-ec8fcfaef122   \n",
       "2437  725f2705-8a6f-1d6e-53b3-142ae21e108c   \n",
       "2438  c06b0d4b-ef84-4a56-0f60-afecab26181c   \n",
       "2439  2255f6bb-0003-4245-3a27-ed67aa101303   \n",
       "\n",
       "                                    parent  \\\n",
       "0     6643d55a-355e-3371-c6b7-45fc96a74513   \n",
       "1                                      NaN   \n",
       "2     15293aeb-7703-3a75-e4d4-bf8028159a0c   \n",
       "3                                      NaN   \n",
       "4     031adf2a-6035-1bd4-1d7e-22db34991c80   \n",
       "...                                    ...   \n",
       "2435  e4970b43-04fd-0362-d73a-bab64815ae3b   \n",
       "2436  c689c679-f02a-3f90-06b1-873a5d94cb53   \n",
       "2437  5a11a546-052a-0a0c-799c-ec8fcfaef122   \n",
       "2438  725f2705-8a6f-1d6e-53b3-142ae21e108c   \n",
       "2439  c06b0d4b-ef84-4a56-0f60-afecab26181c   \n",
       "\n",
       "                                           process_type input_data_type  \n",
       "0                              OneScoredT[Scored[Gene]]      Term[Gene]  \n",
       "1                                             FileInput         FileURL  \n",
       "2                               GeneCountMatrixFromFile             NaN  \n",
       "3                                             FileInput         FileURL  \n",
       "4                               GeneCountMatrixFromFile             NaN  \n",
       "...                                                 ...             ...  \n",
       "2435  ExtractEnrichrTermSearch[LINCS_L1000_Chem_Pert...             NaN  \n",
       "2436                             EnrichrSetTToGMT[Drug]             NaN  \n",
       "2437                                      GenesetsToGMT             GMT  \n",
       "2438                                     GMTConcatenate             NaN  \n",
       "2439                                   SupervennFromGMT             NaN  \n",
       "\n",
       "[2440 rows x 4 columns]"
      ]
     },
     "metadata": {},
     "execution_count": 7
    }
   ],
   "metadata": {}
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "source": [
    "df['input'] = df['process_type'].apply(lambda spec: metanodes.loc[spec, 'input'] if spec in metanodes.index else None)\n",
    "df['output'] = df['process_type'].apply(lambda spec: metanodes.loc[spec, 'output'] if spec in metanodes.index else None)"
   ],
   "outputs": [],
   "metadata": {}
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "source": [
    "for _, edge in df.iterrows():\n",
    "  if edge['process_type'] not in metanode_lookup: continue\n",
    "  process_type_id = metanode_lookup[edge['process_type']]\n",
    "  if edge['output']:\n",
    "    g.add_edge(process_type_id, metanode_lookup[edge['output']])\n",
    "  if edge['input']:\n",
    "    for arg, input_type in json.loads(edge['input']).items():\n",
    "      if type(input_type) == list:\n",
    "        g.add_edge(metanode_lookup[input_type[0]], process_type_id)\n",
    "      else:\n",
    "        g.add_edge(metanode_lookup[input_type], process_type_id)"
   ],
   "outputs": [],
   "metadata": {}
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "source": [
    "nx.write_graphml(g, 'data/output.graphml')"
   ],
   "outputs": [],
   "metadata": {}
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "source": [
    "def n_gram(n, iterable):\n",
    "  gram = []\n",
    "  for item in iterable:\n",
    "    gram.append(item)\n",
    "    if len(gram) == n:\n",
    "      yield tuple(gram)\n",
    "      gram.pop(0)"
   ],
   "outputs": [],
   "metadata": {}
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "source": [
    "with open('data/analytics.tsv', 'r') as fr:\n",
    "  grams = {1: Counter(), 2: Counter(), 3: Counter()}\n",
    "  for hits, *ids in csv.reader(fr, delimiter='\\t'):\n",
    "    for n, c in grams.items():\n",
    "      for g in n_gram(n, ['Start', *ids]):\n",
    "        c.update({ g: int(hits) })"
   ],
   "outputs": [],
   "metadata": {}
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "source": [
    "pd.DataFrame(grams[1].most_common(), columns=['Path', 'Hits'])"
   ],
   "outputs": [
    {
     "output_type": "execute_result",
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Path</th>\n",
       "      <th>Hits</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>(Start,)</td>\n",
       "      <td>7576</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>(FileInput,)</td>\n",
       "      <td>4189</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>(EnrichrSetTToGMT[Phenotype],)</td>\n",
       "      <td>2624</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>(GMTUnion,)</td>\n",
       "      <td>2348</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>(GeneCountMatrixFromFile,)</td>\n",
       "      <td>1907</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>169</th>\n",
       "      <td>(BarplotFrom[Scored[Pathway]],)</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>170</th>\n",
       "      <td>(ExtractEnrichrTermSearch[ARCHS4_Tissues],)</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>171</th>\n",
       "      <td>(GeneTermFromMyVariantInfo,)</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>172</th>\n",
       "      <td>(SomeSetT[Set[Gene]],)</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>173</th>\n",
       "      <td>(Call_MetENP_on_MetSet,)</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>174 rows × 2 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                            Path  Hits\n",
       "0                                       (Start,)  7576\n",
       "1                                   (FileInput,)  4189\n",
       "2                 (EnrichrSetTToGMT[Phenotype],)  2624\n",
       "3                                    (GMTUnion,)  2348\n",
       "4                     (GeneCountMatrixFromFile,)  1907\n",
       "..                                           ...   ...\n",
       "169              (BarplotFrom[Scored[Pathway]],)     1\n",
       "170  (ExtractEnrichrTermSearch[ARCHS4_Tissues],)     1\n",
       "171                 (GeneTermFromMyVariantInfo,)     1\n",
       "172                       (SomeSetT[Set[Gene]],)     1\n",
       "173                     (Call_MetENP_on_MetSet,)     1\n",
       "\n",
       "[174 rows x 2 columns]"
      ]
     },
     "metadata": {},
     "execution_count": 14
    }
   ],
   "metadata": {}
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "source": [
    "pd.DataFrame(grams[2].most_common(), columns=['Path', 'Hits'])"
   ],
   "outputs": [
    {
     "output_type": "execute_result",
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Path</th>\n",
       "      <th>Hits</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>(Start, FileInput)</td>\n",
       "      <td>3095</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>(FileInput, GeneCountMatrixFromFile)</td>\n",
       "      <td>1887</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>(GeneCountMatrixFromFile, TargetRangerScreenTa...</td>\n",
       "      <td>1293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>(FileInput, GeneSigFromFile)</td>\n",
       "      <td>1239</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>(Start, Input[Gene])</td>\n",
       "      <td>1206</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>509</th>\n",
       "      <td>(Input[Set[Gene]], SomeSetT[Set[Gene]])</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>510</th>\n",
       "      <td>(Input[Set[Gene]], GeneshotGeneSetAugmentation...</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>511</th>\n",
       "      <td>(MetaboliteSetInfo, Call_MetENP_on_MetSet)</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>512</th>\n",
       "      <td>(SupervennFromGMT, Input[Gene])</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>513</th>\n",
       "      <td>(MetGeneSearch, GTExTissueExpressionFromGene)</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>514 rows × 2 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                  Path  Hits\n",
       "0                                   (Start, FileInput)  3095\n",
       "1                 (FileInput, GeneCountMatrixFromFile)  1887\n",
       "2    (GeneCountMatrixFromFile, TargetRangerScreenTa...  1293\n",
       "3                         (FileInput, GeneSigFromFile)  1239\n",
       "4                                 (Start, Input[Gene])  1206\n",
       "..                                                 ...   ...\n",
       "509            (Input[Set[Gene]], SomeSetT[Set[Gene]])     1\n",
       "510  (Input[Set[Gene]], GeneshotGeneSetAugmentation...     1\n",
       "511         (MetaboliteSetInfo, Call_MetENP_on_MetSet)     1\n",
       "512                    (SupervennFromGMT, Input[Gene])     1\n",
       "513      (MetGeneSearch, GTExTissueExpressionFromGene)     1\n",
       "\n",
       "[514 rows x 2 columns]"
      ]
     },
     "metadata": {},
     "execution_count": 16
    }
   ],
   "metadata": {}
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "source": [
    "pd.DataFrame(grams[3].most_common(), columns=['Path', 'Hits'])"
   ],
   "outputs": [
    {
     "output_type": "execute_result",
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Path</th>\n",
       "      <th>Hits</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>(Start, FileInput, GeneCountMatrixFromFile)</td>\n",
       "      <td>1802</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>(FileInput, GeneCountMatrixFromFile, TargetRan...</td>\n",
       "      <td>1293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>(GeneCountMatrixFromFile, TargetRangerScreenTa...</td>\n",
       "      <td>963</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>(Start, Input[Pathway], EnrichrTermSearch[Path...</td>\n",
       "      <td>795</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>(Start, FileInput, GeneSigFromFile)</td>\n",
       "      <td>693</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>685</th>\n",
       "      <td>(Start, Input[Set[Gene]], SomeSetT[Set[Gene]])</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>686</th>\n",
       "      <td>(Start, Input[Set[Gene]], GeneshotGeneSetAugme...</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>687</th>\n",
       "      <td>(MGMetTable2MetSet, MetaboliteSetInfo, Call_Me...</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>688</th>\n",
       "      <td>(GMTConcatenate, SupervennFromGMT, Input[Gene])</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>689</th>\n",
       "      <td>(Input[Gene], MetGeneSearch, GTExTissueExpress...</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>690 rows × 2 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                  Path  Hits\n",
       "0          (Start, FileInput, GeneCountMatrixFromFile)  1802\n",
       "1    (FileInput, GeneCountMatrixFromFile, TargetRan...  1293\n",
       "2    (GeneCountMatrixFromFile, TargetRangerScreenTa...   963\n",
       "3    (Start, Input[Pathway], EnrichrTermSearch[Path...   795\n",
       "4                  (Start, FileInput, GeneSigFromFile)   693\n",
       "..                                                 ...   ...\n",
       "685     (Start, Input[Set[Gene]], SomeSetT[Set[Gene]])     1\n",
       "686  (Start, Input[Set[Gene]], GeneshotGeneSetAugme...     1\n",
       "687  (MGMetTable2MetSet, MetaboliteSetInfo, Call_Me...     1\n",
       "688    (GMTConcatenate, SupervennFromGMT, Input[Gene])     1\n",
       "689  (Input[Gene], MetGeneSearch, GTExTissueExpress...     1\n",
       "\n",
       "[690 rows x 2 columns]"
      ]
     },
     "metadata": {},
     "execution_count": 17
    }
   ],
   "metadata": {}
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "source": [
    "d = pd.DataFrame(grams[2].most_common(), columns=['Path', 'Hits'])\n",
    "d['Head'] = d['Path'].apply(lambda p: p[-1])\n",
    "d['Tail'] = d['Path'].apply(lambda p: p[0])\n",
    "_2grams = {tail: { row['Head']: row['Hits'] for _, row in records.iterrows() } for tail, records in d.groupby('Tail')}"
   ],
   "outputs": [],
   "metadata": {}
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "source": [
    "d = pd.DataFrame(grams[3].most_common(), columns=['Path', 'Hits'])\n",
    "d['Head'] = d['Path'].apply(lambda p: p[-1])\n",
    "d['Tail'] = d['Path'].apply(lambda p: ' '.join(p[:-1]))\n",
    "_3grams = {tail: { row['Head']: row['Hits'] for _, row in records.iterrows() } for tail, records in d.groupby('Tail')}"
   ],
   "outputs": [],
   "metadata": {}
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "source": [
    "with open('../app/public/weights.json', 'w') as fw:\n",
    "  json.dump(dict(**_3grams, **_2grams), fw)"
   ],
   "outputs": [],
   "metadata": {}
  }
 ],
 "nbformat": 4,
 "nbformat_minor": 2,
 "metadata": {
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": 3
  },
  "orig_nbformat": 4
 }
}